


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Encoding biomedical knowledge in a computable form">
      
      
        <link rel="canonical" href="https://biological-expression-language.github.io/statements/">
      
      
      <link rel="shortcut icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.1.2, mkdocs-material-5.2.2">
    
    
      
        <title>Statements - Biological Expression Language</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.a2408e81.min.css">
      
        <link rel="stylesheet" href="../assets/stylesheets/palette.a46bcfb3.min.css">
      
      
        
        
        <meta name="theme-color" content="#7e57c2">
      
    
    
    
      
        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&display=fallback">
        <style>body,input{font-family:"Roboto",-apple-system,BlinkMacSystemFont,Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono",SFMono-Regular,Consolas,Menlo,monospace}</style>
      
    
    
    
    
      
        
<link rel="preconnect dns-prefetch" href="https://www.google-analytics.com">
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-166029785-2","https://biological-expression-language.github.io"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){if(this.value){var e=document.location.pathname;ga("send","pageview",e+"?q="+this.value)}})}),document.addEventListener("DOMContentSwitch",function(){ga("send","pageview")})</script>
<script async src="https://www.google-analytics.com/analytics.js"></script>
      
    
    
  </head>
  
  
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="deep-purple" data-md-color-accent="deep-purple">
  
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#entity-modifiers" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
      <header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid" aria-label="Header">
    <a href="https://biological-expression-language.github.io/" title="Biological Expression Language" class="md-header-nav__button md-logo" aria-label="Biological Expression Language">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 003-3 3 3 0 00-3-3 3 3 0 00-3 3 3 3 0 003 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    <label class="md-header-nav__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg>
    </label>
    <div class="md-header-nav__title" data-md-component="header-title">
      
        <div class="md-header-nav__ellipsis">
          <span class="md-header-nav__topic md-ellipsis">
            Biological Expression Language
          </span>
          <span class="md-header-nav__topic md-ellipsis">
            
              Statements
            
          </span>
        </div>
      
    </div>
    
      <label class="md-header-nav__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0116 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 019.5 16 6.5 6.5 0 013 9.5 6.5 6.5 0 019.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
      </label>
      
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" data-md-state="active">
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0116 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 019.5 16 6.5 6.5 0 013 9.5 6.5 6.5 0 019.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </label>
      <button type="reset" class="md-search__icon md-icon" aria-label="Clear" data-md-component="search-reset" tabindex="-1">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg>
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header-nav__source">
        
<a href="https://github.com/biological-expression-language/biological-expression-language.github.io/" title="Go to repository" class="md-source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 00-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 01-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 000 40.81l195.61 195.6a28.86 28.86 0 0040.8 0l194.69-194.69a28.86 28.86 0 000-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    BEL
  </div>
</a>
      </div>
    
  </nav>
</header>
    
    <div class="md-container" data-md-component="container">
      
        
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    <nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="https://biological-expression-language.github.io/" title="Biological Expression Language" class="md-nav__button md-logo" aria-label="Biological Expression Language">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 003-3 3 3 0 00-3-3 3 3 0 00-3 3 3 3 0 003 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    Biological Expression Language
  </label>
  
    <div class="md-nav__source">
      
<a href="https://github.com/biological-expression-language/biological-expression-language.github.io/" title="Go to repository" class="md-source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 00-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 01-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 000 40.81l195.61 195.6a28.86 28.86 0 0040.8 0l194.69-194.69a28.86 28.86 0 000-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    BEL
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  <li class="md-nav__item">
    <a href=".." title="Home" class="md-nav__link">
      Home
    </a>
  </li>

    
      
      
      

  


  <li class="md-nav__item md-nav__item--active md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-2" type="checkbox" id="nav-2" checked>
    
    <label class="md-nav__link" for="nav-2">
      Tutorial
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Tutorial" data-md-level="1">
      <label class="md-nav__title" for="nav-2">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Tutorial
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorial/" title="Overview" class="md-nav__link">
      Overview
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../identifiers/" title="Identifiers" class="md-nav__link">
      Identifiers
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-2-3" type="checkbox" id="nav-2-3">
    
    <label class="md-nav__link" for="nav-2-3">
      Entities
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Entities" data-md-level="2">
      <label class="md-nav__title" for="nav-2-3">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Entities
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../entities/" title="Overview" class="md-nav__link">
      Overview
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../entities/physical/" title="Physical Entities" class="md-nav__link">
      Physical Entities
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../entities/process/" title="Process Entities" class="md-nav__link">
      Process Entities
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../entities/reified/" title="Reified Entities" class="md-nav__link">
      Reified Entities
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-2-4" type="checkbox" id="nav-2-4">
    
    <label class="md-nav__link" for="nav-2-4">
      Relations
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Relations" data-md-level="2">
      <label class="md-nav__title" for="nav-2-4">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Relations
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/" title="Overview" class="md-nav__link">
      Overview
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/causal/" title="Causal" class="md-nav__link">
      Causal
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/correlative/" title="Correlative" class="md-nav__link">
      Correlative
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/associative/" title="Associative" class="md-nav__link">
      Associative
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/genomic/" title="Genomic" class="md-nav__link">
      Genomic
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/ontological/" title="Ontological" class="md-nav__link">
      Ontological
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/deprecated/" title="Deprecated" class="md-nav__link">
      Deprecated
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          

  


  <li class="md-nav__item md-nav__item--active">
    
    <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
    
      
    
    
      <label class="md-nav__link md-nav__link--active" for="__toc">
        Statements
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 9h14V7H3v2m0 4h14v-2H3v2m0 4h14v-2H3v2m16 0h2v-2h-2v2m0-10v2h2V7h-2m0 6h2v-2h-2v2z"/></svg>
        </span>
      </label>
    
    <a href="./" title="Statements" class="md-nav__link md-nav__link--active">
      Statements
    </a>
    
      
<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#edge-modifiers" class="md-nav__link">
    Edge Modifiers
  </a>
  
    <nav class="md-nav" aria-label="Edge Modifiers">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#molecular-activity" class="md-nav__link">
    Molecular Activity
  </a>
  
    <nav class="md-nav" aria-label="Molecular Activity">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#examples" class="md-nav__link">
    Examples
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#transformation-functions" class="md-nav__link">
    Transformation Functions
  </a>
  
    <nav class="md-nav" aria-label="Transformation Functions">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#translocations" class="md-nav__link">
    Translocations
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cell-secretion" class="md-nav__link">
    Cell Secretion
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cell-surface-expression" class="md-nav__link">
    Cell Surface Expression
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#degradation" class="md-nav__link">
    Degradation
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cellular-location" class="md-nav__link">
    Cellular Location
  </a>
  
    <nav class="md-nav" aria-label="Cellular Location">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example-endoplasmic-reticulum-pool-of-ca2" class="md-nav__link">
    Example - Endoplasmic Reticulum pool of Ca^2+^
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nested-statements" class="md-nav__link">
    Nested Statements
  </a>
  
</li>
      
    </ul>
  
</nav>
    
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../annotations/" title="Annotations" class="md-nav__link">
      Annotations
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../cookbook/" title="Cookbook" class="md-nav__link">
      Cookbook
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../changelog/" title="Changelog" class="md-nav__link">
      Changelog
    </a>
  </li>

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#edge-modifiers" class="md-nav__link">
    Edge Modifiers
  </a>
  
    <nav class="md-nav" aria-label="Edge Modifiers">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#molecular-activity" class="md-nav__link">
    Molecular Activity
  </a>
  
    <nav class="md-nav" aria-label="Molecular Activity">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#examples" class="md-nav__link">
    Examples
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#transformation-functions" class="md-nav__link">
    Transformation Functions
  </a>
  
    <nav class="md-nav" aria-label="Transformation Functions">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#translocations" class="md-nav__link">
    Translocations
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cell-secretion" class="md-nav__link">
    Cell Secretion
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#cell-surface-expression" class="md-nav__link">
    Cell Surface Expression
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#degradation" class="md-nav__link">
    Degradation
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cellular-location" class="md-nav__link">
    Cellular Location
  </a>
  
    <nav class="md-nav" aria-label="Cellular Location">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example-endoplasmic-reticulum-pool-of-ca2" class="md-nav__link">
    Example - Endoplasmic Reticulum pool of Ca^2+^
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#nested-statements" class="md-nav__link">
    Nested Statements
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                  <a href="https://github.com/biological-expression-language/biological-expression-language.github.io/edit/master/docs/statements.md" title="Edit this page" class="md-content__button md-icon">
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"/></svg>
                  </a>
                
                
                  
                
                
                <h1 id="entity-modifiers">Entity Modifiers<a class="headerlink" href="#entity-modifiers" title="Permanent link">&para;</a></h1>
<p>While BEL terms and relations are the building bocks for BEL statements, there are a few things that can be added to make them much more rich.</p>
<h2 id="edge-modifiers">Edge Modifiers<a class="headerlink" href="#edge-modifiers" title="Permanent link">&para;</a></h2>
<h3 id="molecular-activity">Molecular Activity<a class="headerlink" href="#molecular-activity" title="Permanent link">&para;</a></h3>
<p>You might have noticed functions for representing chemicals, genes, RNAs, proteins, and all have the suffix <code>abundance</code>. Sometimes it makes more sense to talk about the activity of a physical entity being responsible (as subject) or modified (as object).</p>
<p>The <code>activity(&lt;abundance&gt;)</code> or <code>act(&lt;abundance)</code> is used to specify events resulting from the molecular activity of an abundance. The <code>activity()</code> function provides distinct terms that enable differentiation of the increase or decrease of the molecular activity of a protein from changes in the abundance of the protein. <code>activity()</code> can be applied to a protein, complex, or RNA abundance term, and modified with a <code>`molecularActivity()</code> argument to indicate a specific type of molecular activity.</p>
<div class="highlight"><pre><span></span><code># long form
activity(p(hgnc:391 ! :AKT1))

# short form
act(p(hgnc:391 ! :AKT1))
</code></pre></div>

<p>To more specifically talk about what kind of activity a physical entity has, you can use the <code>molecularActivity()</code> or <code>ma()</code>. It comes after the BEL term inside the <code>act()</code> like below</p>
<div class="highlight"><pre><span></span><code># long form
act(p(hgnc:391 ! AKT1), molecularActivity(kin))

# short form
act(p(hgnc:391 ! AKT1), ma(kin))

# using equivalent Gene Ontology
act(p(hgnc:391 ! AKT1), ma(go:0016301 ! &quot;kinase activity&quot;))
</code></pre></div>

<p>In general, the <code>ma()</code> function can be used like:</p>
<div class="highlight"><pre><span></span><code>ma(&lt;default term&gt;)

# OR

ma(prefix:identifier [! name])
</code></pre></div>

<p>The BEL default namespace has the following commonly used molecular activities, but it makes most sense to map these to entries in the Gene Ontology Molecular Function namespace.</p>
<table>
<thead>
<tr>
<th align="left">short</th>
<th align="left">long</th>
<th align="left">Gene Ontology</th>
<th align="left">GO Label</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left">cat</td>
<td align="left">catalyticActivity</td>
<td align="left">0003824</td>
<td align="left">catalytic activity</td>
</tr>
<tr>
<td align="left">chap</td>
<td align="left">chaperoneActivity</td>
<td align="left">0044183</td>
<td align="left">protein binding involved in protein folding</td>
</tr>
<tr>
<td align="left">gap</td>
<td align="left">gapActivity</td>
<td align="left">0032794</td>
<td align="left">GTPase activating protein binding</td>
</tr>
<tr>
<td align="left">gef</td>
<td align="left">gefActivity</td>
<td align="left">0005085</td>
<td align="left">guanyl-nucleotide exchange factor activity</td>
</tr>
<tr>
<td align="left">gtp</td>
<td align="left">gtpBoundActivity</td>
<td align="left">0005525</td>
<td align="left">GTP binding</td>
</tr>
<tr>
<td align="left">kin</td>
<td align="left">kinaseActivity</td>
<td align="left">0016301</td>
<td align="left">kinase activity</td>
</tr>
<tr>
<td align="left">pep</td>
<td align="left">peptidaseActivity</td>
<td align="left">0008233</td>
<td align="left">peptidase activity</td>
</tr>
<tr>
<td align="left">phos</td>
<td align="left">phosphotaseActivity</td>
<td align="left">0016791</td>
<td align="left">phosphatase activity</td>
</tr>
<tr>
<td align="left">ribo</td>
<td align="left">ribosylationActivity</td>
<td align="left">0003956</td>
<td align="left">NAD(P)+-protein-arginine ADP-ribosyltransferase activity</td>
</tr>
<tr>
<td align="left">tscript</td>
<td align="left">transcriptionalActivity</td>
<td align="left">0001071</td>
<td align="left">nucleic acid binding transcription factor activity</td>
</tr>
<tr>
<td align="left">tport</td>
<td align="left">transportActivity</td>
<td align="left">0005215</td>
<td align="left">transporter activity</td>
</tr>
</tbody>
</table>
<h4 id="examples">Examples<a class="headerlink" href="#examples" title="Permanent link">&para;</a></h4>
<p><strong>default BEL namespace, transcriptional activity (DEFAULT namespace is optional)</strong></p>
<div class="highlight"><pre><span></span><code>act(p(HGNC:FOXO1), ma(tscript))
</code></pre></div>

<p><strong>GO molecular function namespace, transcriptional activity</strong></p>
<div class="highlight"><pre><span></span><code>act(p(HGNC:FOXO1), ma(GO:&quot;nucleic acid binding transcription factor activity&quot;))
</code></pre></div>

<p><strong>default BEL namespace, kinase activity</strong></p>
<div class="highlight"><pre><span></span><code>act(p(HGNC:AKT1), ma(kin))
</code></pre></div>

<p><strong>GO molecular function namespace, kinase activity</strong></p>
<div class="highlight"><pre><span></span><code>act(p(HGNC:AKT1), ma(GO:&quot;kinase activity&quot;))
</code></pre></div>

<p><strong>Non-Specified Activities</strong></p>
<p>If the type of molecular activity is not reported, it does not need to be specified. The <code>activity()</code> function is sufficient for distinguishing the frequency of events mediated by an abundance from the amount of the abundance. This term represents the ligand-bound activity of the human non-catalytic receptor protein TLR7.</p>
<div class="highlight"><pre><span></span><code># long form
activity(proteinAbundance(HGNC:TLR7))

# short form
act(p(HGNC:TLR7))
</code></pre></div>

<p><strong>Catalytic Activity</strong></p>
<p>A protein, complex, or ribozymes has catalytic activity when it acts as an enzymatic catalyst of biochemical reactions. Catalytic activity includes kinase, phosphatase, peptidase, and ADP-ribosylase activities, though these can be represented by more specific molecular activity terms.</p>
<p>This term represents the frequency of events in which the protein abundance of rat Sod1 acts as a catalyst.</p>
<div class="highlight"><pre><span></span><code># Using BEL default namespace
act(p(RGD:Sod1), ma(cat))

# Using Gene Ontology
act(p(RGD:Sod1), ma(GO:&quot;catalytic activity&quot;))
</code></pre></div>

<p><strong>Peptidase Activity</strong></p>
<p>This term represents the frequency of events in which the protein abundance of mouse Casp3 acts as a peptidase.The more specific GO Molecular Function term "cysteine-type endopeptidase activity" is also applicable.</p>
<div class="highlight"><pre><span></span><code># Using BEL default namespace
act(p(MGI:Casp3), ma(pep))

# Using Gene Ontology
act(p(MGI:Casp3), ma(GO:&quot;peptidase activity&quot;))
</code></pre></div>

<p><strong>G-proteins in the active (GTP-bound) state</strong></p>
<p>The activity of guanine nucleotide-binding proteins (G-proteins) like RAS in the active, GTP-bound state. This term represents the frequency of events caused by the active, GTP-bound form of the RAS protein family.</p>
<div class="highlight"><pre><span></span><code># Using BEL default namespace
act(p(SFAM:&quot;RAS Family&quot;), ma(gtp))

# Using Gene Ontology
act(p(SFAM:&quot;RAS Family&quot;), ma(GO:&quot;GTP binding&quot;))
</code></pre></div>

<p><strong>Transporter Activity</strong></p>
<p>Molecular translocation events mediated by transporter proteins like ion channels or glucose transporters. This term represents the frequency of ion transport events mediated by the epithelial sodium channel (ENaC) complex.</p>
<div class="highlight"><pre><span></span><code># Using BEL default namespace
act(complex(NCH:&quot;ENaC Complex&quot;), ma(tport))

# Using Gene Ontology
act(complex(NCH:&quot;ENaC Complex&quot;), ma(GO:&quot;transporter activity&quot;))
</code></pre></div>

<p><strong>Chaperone Activity</strong></p>
<p>This term represents the events in which the human Calnexin protein functions as a chaperone to aid the folding of other proteins.</p>
<div class="highlight"><pre><span></span><code># Using BEL default namespace
act(p(HGNC:CANX), ma(chap))

# Using Gene Ontology
</code></pre></div>

<p><strong>Transcription Activity</strong></p>
<p>Events in which a protein or molecular complex acts to directly control transcription, including proteins acting directly as transcription factors, as well as transcriptional co-activators and co-repressors. This term represents the frequency of events in which the mouse p53 protein controls RNA expression.</p>
<div class="highlight"><pre><span></span><code># Using BEL default namespace
act(p(MGI:Trp53), ma(tscript))

# Using Gene Ontology 
act(p(MGI:Trp53), ma(GO:&quot;nucleic acid binding transcription factor activity&quot;))
</code></pre></div>

<h2 id="transformation-functions">Transformation Functions<a class="headerlink" href="#transformation-functions" title="Permanent link">&para;</a></h2>
<p>The following BEL functions represent parametrized biological processes. Currently, BEL parametrizes translocation and degradation processes.</p>
<h3 id="translocations">Translocations<a class="headerlink" href="#translocations" title="Permanent link">&para;</a></h3>
<p>A <code>translocation(&lt;abundance&gt;, fromLocation(ns1:v1), toLocation(ns2:v2))</code> or <code>tloc(&lt;abundance&gt;, fromLoc(ns1:v1), toLoc(ns2:v2))</code> denotes the process in which a physical entity is moved from one location to another.</p>
<p>For example, endocytosis (translocation from the cell surface to the endosome) of the epidermal growth factor receptor (EGFR) protein can be represented as:</p>
<div class="highlight"><pre><span></span><code># Long form
translocation(p(hgnc:3236 ! EGFR), fromLocation(go:0009986 ! &quot;cell surface&quot;), toLocaction(go:0005768 ! endosome))

# Short form
tloc(p(hgnc:3236 ! EGFR), fromLoc(go:0009986 ! &quot;cell surface&quot;), toLoc(go:0005768 ! endosome))
</code></pre></div>

<p>In general, the <code>fromLoc()</code> and <code>toLoc()</code> functions take in an identifier:</p>
<div class="highlight"><pre><span></span><code>toLoc(prefix:identifier [! name])
fromLoc(prefix:identifier [! name])
</code></pre></div>

<p>The most reasonable namespaces to use for locations are from the GO cellular components branch and maybe cell types. MeSH also contains locations, but are not recommended because of lack of mappings.</p>
<h3 id="cell-secretion">Cell Secretion<a class="headerlink" href="#cell-secretion" title="Permanent link">&para;</a></h3>
<p>This term represents the event in which human NFE2L2 protein is translocated from the cytoplasm to the nucleus.</p>
<div class="highlight"><pre><span></span><code># long form
translocation(p(HGNC:NFE2L2), fromLoc(mesh:D003593 ! Cytoplasm), toLoc(mesh:D002467 ! &quot;Cell Nucleus&quot;))

# short form
tloc(p(HGNC:NFE2L2), fromLoc(mesh:D003593 ! Cytoplasm), toLoc(mesh:D002467 ! &quot;Cell Nucleus&quot;))
</code></pre></div>

<p>There are also two convenience functions, <code>cellSurfaceExpression()</code> and <code>cellSecretion()</code>, that are so common that they are part of BEL with pre-defined <code>toLoc()</code> and <code>fromLoc()</code>.</p>
<p>For example, the following represents secretion of mouse IL6 protein.</p>
<div class="highlight"><pre><span></span><code># long form
cellSecretion(p(mgi:96559 ! Il6))

# short form
sec(p(mgi:96559 ! Il6))
</code></pre></div>

<h3 id="cell-surface-expression">Cell Surface Expression<a class="headerlink" href="#cell-surface-expression" title="Permanent link">&para;</a></h3>
<p><code>cellSurfaceExpression(&lt;abundance&gt;)</code> or <code>surf(&lt;abundance&gt;)</code> denotes the frequency or abundance of events in which members of <code>&lt;abundance&gt;</code> move to the surface of cells. <code>cellSurfaceExpression(&lt;abundance&gt;)</code> can be equivalently expressed as:</p>
<div class="highlight"><pre><span></span><code>tloc(&lt;abundance&gt;, fromLoc(go:intracellular), toLoc(go:&quot;cell surface&quot;))
</code></pre></div>

<p>The intent of the <code>cellSurfaceExpression()</code> function is to provide a simple, standard means of expressing a commonly represented translocation.</p>
<p>For the abundance term A, <code>cellSecretion(&lt;abundance&gt;)</code> or <code>sec(&lt;abundance&gt;)</code> denotes the frequency or number of events in which members of <code>&lt;abundance&gt;</code> move from cells to regions outside of the cells. <code>cellSecretion(&lt;abundance&gt;</code> can be equivalently expressed as:</p>
<div class="highlight"><pre><span></span><code>tloc(&lt;abundance&gt;, fromLoc(go:intracellular), toLoc(go:&quot;extracellular space&quot;))
</code></pre></div>

<p>The intent of the <code>cellSecretion()</code> function is to provide a simple, standard means of expressing a commonly represented translocation.</p>
<p>This term represents cell surface expression of rat Fas protein.</p>
<div class="highlight"><pre><span></span><code># long form
cellSurfaceExpression(p(rgd:619831 ! Fas))

# short form
surf(p(rgd:619831 ! Fas))
</code></pre></div>

<h3 id="degradation">Degradation<a class="headerlink" href="#degradation" title="Permanent link">&para;</a></h3>
<p>Events in which an abundance is degraded can be represented by the <code>degradation()</code> or <code>deg()</code> function.</p>
<p><code>degradation(&lt;abundance&gt;)</code> or <code>deg(&lt;abundance&gt;)</code> denotes the frequency or number of events in which a member of <code>&lt;abundance&gt;</code> is degraded in some way such that it is no longer a member of <code>&lt;abundance&gt;</code>. For example, <code>degradation()</code> is used to represent proteasome-mediated proteolysis.</p>
<p>If can be inferred that:</p>
<div class="highlight"><pre><span></span><code>deg(X) directlyDecreases X
</code></pre></div>

<p>This term represents the degradation of MYC RNA. Degradation decreases the amount of the abundance - when degradation statements are compiled, a directlyDecreases relationship edge is added between the degradation term and the degraded entity.</p>
<div class="highlight"><pre><span></span><code># long form
degradation(rnaAbundance(hgnc:7553 ! MYC))

# short form
deg(r(hgnc:7553 ! MYC))
</code></pre></div>

<p><strong>Note</strong> the degradation relationship is not recommended, because it's more direct to just use the <code>decreases</code> relationship to show that the abundance is lowered.</p>
<h2 id="cellular-location">Cellular Location<a class="headerlink" href="#cellular-location" title="Permanent link">&para;</a></h2>
<p>Unlike the translocation process, each node can be annotated with the location where it is inside the cell with the <code>location()</code> / <code>loc()</code> function. It can be used as an argument within any abundance function except <code>compositeAbundance()</code> to represent a distinct subset of the abundance at that location. Location subsets of abundances have the general form:</p>
<div class="highlight"><pre><span></span><code>f(ns:v, loc(ns:v))
</code></pre></div>

<p>The location should come following all modifications and variants.</p>
<p>For example, the AKT protein in the cytoplasm can be encoded in BEL as:</p>
<div class="highlight"><pre><span></span><code># long form
p(hgnc:391 ! AKT1, location(mesh:D003593 ! Cytoplasm))

# short form
p(hgnc:391 ! AKT1, loc(mesh:D003593 ! Cytoplasm))
</code></pre></div>

<p>While this information is written in the BEL term that is in either the subject or object, the fact that the entity is in a given location is actually metadata that is stored at the statement leve.</p>
<h4 id="example-endoplasmic-reticulum-pool-of-ca2">Example - Endoplasmic Reticulum pool of Ca^2+^<a class="headerlink" href="#example-endoplasmic-reticulum-pool-of-ca2" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code>a(chebi:29108 ! &quot;calcium(2+)&quot;, loc(go:0005783 ! &quot;endoplasmic reticulum&quot;))
</code></pre></div>

<h2 id="nested-statements">Nested Statements<a class="headerlink" href="#nested-statements" title="Permanent link">&para;</a></h2>
<p>While most BEL statements take the form of <code>&lt;subject&gt; &lt;predicate&gt; &lt;object&gt;</code> where the subject and object may have some of the preceding modifiers attached, it is also possible to use an entire BEL statement as the object. This makes BEL a recursively defined language</p>
<p>Nested statements should only be curated when there is clear transitivity between the two statements and the relationships are casual.</p>
<p>For example, <strong>GATA1</strong> directly increases the process in which the activity of the <strong>ZBTB16</strong> protein directly increases the abundance of RNA designated by <strong>MPL</strong>. This can be encoded in BEL as:</p>
<div class="highlight"><pre><span></span><code>p(hgnc:4170 ! GATA1) =&gt; ( act(p(hgnc:12930 ! ZBTB16)) =&gt; r(hgnc:7217 ! MPL) )
</code></pre></div>

<p>If you are working in a network context, nested statements in the form of</p>
<div class="highlight"><pre><span></span><code>A P1 (B P2 C)
</code></pre></div>

<p>can be decomposed into</p>
<div class="highlight"><pre><span></span><code>A P1 B
B P2 C
A P1 * P2 C
</code></pre></div>

<p>Where <code>P1</code> and <code>P2</code> are predicates, and their product <code>P1 * P2</code> is increases if both are increases or both are decreases. Their product <code>P1 * P2</code> is decreases if one is decreases and the other is increases.</p>
<p>Another example: the CLSPN protein is considered to directly activate the phosphorylation of CHEK1 protein by the kinase activity of ATR, because the CLSPN and ATR proteins physically interact.</p>
<div class="highlight"><pre><span></span><code>p(hgnc:19715 ! CLSPN) =&gt; (act(p(hgnc:882 ! ATR), ma(kin)) =&gt; p(hgnc:1925 ! CHEK1, pmod(Ph)))
</code></pre></div>
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
    <div class="md-footer-nav">
      <nav class="md-footer-nav__inner md-grid" aria-label="Footer">
        
          <a href="../relations/deprecated/" title="Deprecated" class="md-footer-nav__link md-footer-nav__link--prev" rel="prev">
            <div class="md-footer-nav__button md-icon">
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
            </div>
            <div class="md-footer-nav__title">
              <div class="md-ellipsis">
                <span class="md-footer-nav__direction">
                  Previous
                </span>
                Deprecated
              </div>
            </div>
          </a>
        
        
          <a href="../annotations/" title="Annotations" class="md-footer-nav__link md-footer-nav__link--next" rel="next">
            <div class="md-footer-nav__title">
              <div class="md-ellipsis">
                <span class="md-footer-nav__direction">
                  Next
                </span>
                Annotations
              </div>
            </div>
            <div class="md-footer-nav__button md-icon">
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"/></svg>
            </div>
          </a>
        
      </nav>
    </div>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
        Made with
        <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
          Material for MkDocs
        </a>
      </div>
      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../assets/javascripts/vendor.d710d30a.min.js"></script>
      <script src="../assets/javascripts/bundle.5f27aba8.min.js"></script><script id="__lang" type="application/json">{"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents"}</script>
      
      <script>
        app = initialize({
          base: "..",
          features: ["instant"],
          search: Object.assign({
            worker: "../assets/javascripts/worker/search.27c6a5e6.min.js"
          }, typeof search !== "undefined" && search)
        })
      </script>
      
    
  </body>
</html>