


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Encoding biomedical knowledge in a computable form">
      
      
        <link rel="canonical" href="https://biological-expression-language.github.io/cookbook/">
      
      
      <link rel="shortcut icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.1.2, mkdocs-material-5.2.2">
    
    
      
        <title>Cookbook - Biological Expression Language</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.a2408e81.min.css">
      
        <link rel="stylesheet" href="../assets/stylesheets/palette.a46bcfb3.min.css">
      
      
        
        
        <meta name="theme-color" content="#7e57c2">
      
    
    
    
      
        <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700%7CRoboto+Mono&display=fallback">
        <style>body,input{font-family:"Roboto",-apple-system,BlinkMacSystemFont,Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono",SFMono-Regular,Consolas,Menlo,monospace}</style>
      
    
    
    
    
      
        
<link rel="preconnect dns-prefetch" href="https://www.google-analytics.com">
<script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-166029785-2","https://biological-expression-language.github.io"),ga("set","anonymizeIp",!0),ga("send","pageview"),document.addEventListener("DOMContentLoaded",function(){document.forms.search&&document.forms.search.query.addEventListener("blur",function(){if(this.value){var e=document.location.pathname;ga("send","pageview",e+"?q="+this.value)}})}),document.addEventListener("DOMContentSwitch",function(){ga("send","pageview")})</script>
<script async src="https://www.google-analytics.com/analytics.js"></script>
      
    
    
  </head>
  
  
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="deep-purple" data-md-color-accent="deep-purple">
  
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#cookbook" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
      <header class="md-header" data-md-component="header">
  <nav class="md-header-nav md-grid" aria-label="Header">
    <a href="https://biological-expression-language.github.io/" title="Biological Expression Language" class="md-header-nav__button md-logo" aria-label="Biological Expression Language">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 003-3 3 3 0 00-3-3 3 3 0 00-3 3 3 3 0 003 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    <label class="md-header-nav__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2z"/></svg>
    </label>
    <div class="md-header-nav__title" data-md-component="header-title">
      
        <div class="md-header-nav__ellipsis">
          <span class="md-header-nav__topic md-ellipsis">
            Biological Expression Language
          </span>
          <span class="md-header-nav__topic md-ellipsis">
            
              Cookbook
            
          </span>
        </div>
      
    </div>
    
      <label class="md-header-nav__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0116 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 019.5 16 6.5 6.5 0 013 9.5 6.5 6.5 0 019.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
      </label>
      
<div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" data-md-state="active">
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0116 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 019.5 16 6.5 6.5 0 013 9.5 6.5 6.5 0 019.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </label>
      <button type="reset" class="md-search__icon md-icon" aria-label="Clear" data-md-component="search-reset" tabindex="-1">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"/></svg>
      </button>
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header-nav__source">
        
<a href="https://github.com/biological-expression-language/biological-expression-language.github.io/" title="Go to repository" class="md-source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 00-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 01-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 000 40.81l195.61 195.6a28.86 28.86 0 0040.8 0l194.69-194.69a28.86 28.86 0 000-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    BEL
  </div>
</a>
      </div>
    
  </nav>
</header>
    
    <div class="md-container" data-md-component="container">
      
        
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              <div class="md-sidebar md-sidebar--primary" data-md-component="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    <nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="https://biological-expression-language.github.io/" title="Biological Expression Language" class="md-nav__button md-logo" aria-label="Biological Expression Language">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 003-3 3 3 0 00-3-3 3 3 0 00-3 3 3 3 0 003 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54z"/></svg>

    </a>
    Biological Expression Language
  </label>
  
    <div class="md-nav__source">
      
<a href="https://github.com/biological-expression-language/biological-expression-language.github.io/" title="Go to repository" class="md-source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M439.55 236.05L244 40.45a28.87 28.87 0 00-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 01-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 000 40.81l195.61 195.6a28.86 28.86 0 0040.8 0l194.69-194.69a28.86 28.86 0 000-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    BEL
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      


  <li class="md-nav__item">
    <a href=".." title="Home" class="md-nav__link">
      Home
    </a>
  </li>

    
      
      
      

  


  <li class="md-nav__item md-nav__item--active md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-2" type="checkbox" id="nav-2" checked>
    
    <label class="md-nav__link" for="nav-2">
      Tutorial
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Tutorial" data-md-level="1">
      <label class="md-nav__title" for="nav-2">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Tutorial
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../tutorial/" title="Tutorial" class="md-nav__link">
      Tutorial
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../identifiers/" title="Identifiers" class="md-nav__link">
      Identifiers
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-2-3" type="checkbox" id="nav-2-3">
    
    <label class="md-nav__link" for="nav-2-3">
      Entities
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Entities" data-md-level="2">
      <label class="md-nav__title" for="nav-2-3">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Entities
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../entities/" title="Overview" class="md-nav__link">
      Overview
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../entities/physical/" title="Physical Entities" class="md-nav__link">
      Physical Entities
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../entities/process/" title="Process Entities" class="md-nav__link">
      Process Entities
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../entities/reified/" title="Reified Entities" class="md-nav__link">
      Reified Entities
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item md-nav__item--nested">
    
      <input class="md-nav__toggle md-toggle" data-md-toggle="nav-2-4" type="checkbox" id="nav-2-4">
    
    <label class="md-nav__link" for="nav-2-4">
      Relations
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.59 16.58L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.42z"/></svg>
      </span>
    </label>
    <nav class="md-nav" aria-label="Relations" data-md-level="2">
      <label class="md-nav__title" for="nav-2-4">
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
        </span>
        Relations
      </label>
      <ul class="md-nav__list" data-md-scrollfix>
        
        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/" title="Overview" class="md-nav__link">
      Overview
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/causal/" title="Causal" class="md-nav__link">
      Causal
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/correlative/" title="Correlative" class="md-nav__link">
      Correlative
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/associative/" title="Associative" class="md-nav__link">
      Associative
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/genomic/" title="Genomic" class="md-nav__link">
      Genomic
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/ontological/" title="Ontological" class="md-nav__link">
      Ontological
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../relations/deprecated/" title="Deprecated" class="md-nav__link">
      Deprecated
    </a>
  </li>

        
      </ul>
    </nav>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../statements/" title="Statements" class="md-nav__link">
      Statements
    </a>
  </li>

        
          
          
          


  <li class="md-nav__item">
    <a href="../annotations/" title="Annotations" class="md-nav__link">
      Annotations
    </a>
  </li>

        
          
          
          

  


  <li class="md-nav__item md-nav__item--active">
    
    <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
    
      
    
    
      <label class="md-nav__link md-nav__link--active" for="__toc">
        Cookbook
        <span class="md-nav__icon md-icon">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 9h14V7H3v2m0 4h14v-2H3v2m0 4h14v-2H3v2m16 0h2v-2h-2v2m0-10v2h2V7h-2m0 6h2v-2h-2v2z"/></svg>
        </span>
      </label>
    
    <a href="./" title="Cookbook" class="md-nav__link md-nav__link--active">
      Cookbook
    </a>
    
      
<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#representation-of-experimental-data" class="md-nav__link">
    Representation of Experimental Data
  </a>
  
    <nav class="md-nav" aria-label="Representation of Experimental Data">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#subject-terms-perturbations" class="md-nav__link">
    Subject Terms (Perturbations)
  </a>
  
    <nav class="md-nav" aria-label="Subject Terms (Perturbations)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#belv2-how-should-i-represent-chemical-inhibitor-experiments" class="md-nav__link">
    BELv2 How should I represent chemical inhibitor experiments?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-experiments-that-use-site-directed-mutants" class="md-nav__link">
    How do I represent experiments that use site-directed mutants?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#non-phosphorylatable-mutant" class="md-nav__link">
    Non-phosphorylatable mutant
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-observations-resulting-from-manipulation-of-two-or-more-entities" class="md-nav__link">
    How do I represent observations resulting from manipulation of two or more entities?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-should-i-represent-gene-knock-out-or-rnai-experiments" class="md-nav__link">
    How should I represent gene knock out or RNAi experiments?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gene-knockouts" class="md-nav__link">
    Gene knockouts
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rna-interference" class="md-nav__link">
    RNA interference
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-should-i-represent-overexpression-experiments" class="md-nav__link">
    How should I represent overexpression experiments?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#when-should-i-use-the-protein-abundance-vs-the-activity-of-a-protein" class="md-nav__link">
    When should I use the protein abundance vs. the activity of a protein?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#effects-are-not-due-to-the-catalytic-activity-of-the-protein" class="md-nav__link">
    Effects are not due to the catalytic activity of the protein
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#effects-are-likely-due-to-the-activity-of-the-protein" class="md-nav__link">
    Effects are likely due to the activity of the protein
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#not-enough-information-is-available" class="md-nav__link">
    Not enough information is available
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#relationships" class="md-nav__link">
    Relationships
  </a>
  
    <nav class="md-nav" aria-label="Relationships">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#when-should-i-use-a-correlative-relationship" class="md-nav__link">
    When should I use a correlative relationship?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#correlative" class="md-nav__link">
    Correlative
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#causal" class="md-nav__link">
    Causal
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#object-terms-measurements" class="md-nav__link">
    Object Terms (Measurements)
  </a>
  
    <nav class="md-nav" aria-label="Object Terms (Measurements)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#how-should-i-represent-microarray-data" class="md-nav__link">
    How should I represent microarray data?
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#statement-annotations" class="md-nav__link">
    Statement Annotations
  </a>
  
    <nav class="md-nav" aria-label="Statement Annotations">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#how-do-i-annotate-a-relationship-observed-in-multiple-biological-contexts" class="md-nav__link">
    How do I annotate a relationship observed in multiple biological contexts?
  </a>
  
    <nav class="md-nav" aria-label="How do I annotate a relationship observed in multiple biological contexts?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example" class="md-nav__link">
    Example
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#modified-proteins" class="md-nav__link">
    Modified Proteins
  </a>
  
    <nav class="md-nav" aria-label="Modified Proteins">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-a-protein-modification-when-specific-information-is-not-available" class="md-nav__link">
    How do I represent a protein modification when specific information is not available?
  </a>
  
    <nav class="md-nav" aria-label="How do I represent a protein modification when specific information is not available?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example_1" class="md-nav__link">
    Example
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example_2" class="md-nav__link">
    Example
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-a-protein-modification-within-a-complex" class="md-nav__link">
    How do I represent a protein modification within a complex?
  </a>
  
    <nav class="md-nav" aria-label="How do I represent a protein modification within a complex?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#exception-modified-histones-bound-to-the-promoter-of-a-specific-gene" class="md-nav__link">
    Exception – modified histones bound to the promoter of a specific gene
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-a-situation-where-multiple-phosphorylations-are-required-for-a-proteins-activity" class="md-nav__link">
    How do I represent a situation where multiple phosphorylations are required for a protein's activity?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-a-situation-where-one-protein-modification-initiates-additional-modifications" class="md-nav__link">
    How do I represent a situation where one protein modification initiates additional modifications?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-removal-of-a-protein-modification-eg-dephosphorylation-deubiquitination" class="md-nav__link">
    How do I represent removal of a protein modification (e.g., dephosphorylation, deubiquitination)?
  </a>
  
    <nav class="md-nav" aria-label="How do I represent removal of a protein modification (e.g., dephosphorylation, deubiquitination)?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#deubiquitination" class="md-nav__link">
    Deubiquitination
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dephosphorylation" class="md-nav__link">
    Dephosphorylation
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#reactions" class="md-nav__link">
    Reactions
  </a>
  
    <nav class="md-nav" aria-label="Reactions">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#how-can-i-represent-a-reversible-metabolic-reaction" class="md-nav__link">
    How can I represent a reversible metabolic reaction?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#when-and-why-should-i-use-a-reaction-term" class="md-nav__link">
    When and why should I use a reaction term?
  </a>
  
    <nav class="md-nav" aria-label="When and why should I use a reaction term?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#when_-should-i-use-a-reaction-term" class="md-nav__link">
    When_ should I use a reaction term
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#why_-should-i-use-a-reaction-term" class="md-nav__link">
    Why_ should I use a reaction term
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#protein-protein-interactions" class="md-nav__link">
    Protein-Protein Interactions
  </a>
  
    <nav class="md-nav" aria-label="Protein-Protein Interactions">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-a-physical-interaction-between-two-entities" class="md-nav__link">
    How do I represent a physical interaction between two entities?
  </a>
  
    <nav class="md-nav" aria-label="How do I represent a physical interaction between two entities?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#examples" class="md-nav__link">
    Examples
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#protein-families" class="md-nav__link">
    Protein Families
  </a>
  
    <nav class="md-nav" aria-label="Protein Families">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#when-should-i-use-a-protein-family-instead-of-a-specific-protein" class="md-nav__link">
    When should I use a protein family instead of a specific protein?
  </a>
  
    <nav class="md-nav" aria-label="When should I use a protein family instead of a specific protein?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example-1" class="md-nav__link">
    Example 1:
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-2" class="md-nav__link">
    Example 2:
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-3" class="md-nav__link">
    Example 3:
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#more-cookbook" class="md-nav__link">
    More Cookbook
  </a>
  
    <nav class="md-nav" aria-label="More Cookbook">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#causal-statement-examples" class="md-nav__link">
    Causal Statement Examples
  </a>
  
    <nav class="md-nav" aria-label="Causal Statement Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#causal-increase" class="md-nav__link">
    Causal increase
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example_3" class="md-nav__link">
    Example
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#causal-decrease" class="md-nav__link">
    Causal decrease
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example_4" class="md-nav__link">
    Example
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_1" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_1" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#causes-no-change" class="md-nav__link">
    Causes no change
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example_5" class="md-nav__link">
    Example
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_2" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_2" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#correlative-statement-examples" class="md-nav__link">
    Correlative Statement Examples
  </a>
  
    <nav class="md-nav" aria-label="Correlative Statement Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#negativecorrelation" class="md-nav__link">
    negativeCorrelation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#association" class="md-nav__link">
    association
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_3" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_3" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#direct-causal-statement-examples" class="md-nav__link">
    Direct Causal Statement Examples
  </a>
  
    <nav class="md-nav" aria-label="Direct Causal Statement Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example-ligand-and-receptor" class="md-nav__link">
    Example - Ligand and Receptor
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_4" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_4" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-kinase-and-substrate" class="md-nav__link">
    Example - Kinase and Substrate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_5" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-catalyst-and-reaction" class="md-nav__link">
    Example - Catalyst and Reaction
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_6" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_5" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-self-referential-relationships" class="md-nav__link">
    Example - Self-Referential Relationships
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_7" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_6" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-direct-transcriptional-control" class="md-nav__link">
    Example - Direct Transcriptional Control
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_8" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_7" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nested-statement-example" class="md-nav__link">
    Nested Statement Example
  </a>
  
    <nav class="md-nav" aria-label="Nested Statement Example">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#long-form_9" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_8" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#other-examples" class="md-nav__link">
    Other Examples
  </a>
  
    <nav class="md-nav" aria-label="Other Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#membership-assignment-examples" class="md-nav__link">
    Membership Assignment Examples
  </a>
  
    <nav class="md-nav" aria-label="Membership Assignment Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#protein-family" class="md-nav__link">
    Protein Family
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#complex-component" class="md-nav__link">
    Complex Component
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-binding-interaction" class="md-nav__link">
    Example - Binding Interaction
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-protein-protein-interaction-as-bel-statement" class="md-nav__link">
    Example - protein-protein interaction as BEL statement
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-protein-protein-interaction-as-statement-object" class="md-nav__link">
    Example - protein-protein interaction as Statement object
  </a>
  
    <nav class="md-nav" aria-label="Example - protein-protein interaction as Statement object">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#protein-dna-interactions" class="md-nav__link">
    Protein – DNA interactions
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-transcription-factor-protein-binding-to-dna" class="md-nav__link">
    Example - transcription factor protein binding to DNA
  </a>
  
    <nav class="md-nav" aria-label="Example - transcription factor protein binding to DNA">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#protein-small-molecule-interactions" class="md-nav__link">
    Protein – small molecule interactions
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-protein-binding-to-a-small-molecule" class="md-nav__link">
    Example - protein binding to a small molecule
  </a>
  
    <nav class="md-nav" aria-label="Example - protein binding to a small molecule">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#causal_1" class="md-nav__link">
    Causal
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#causal_2" class="md-nav__link">
    Causal
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
    
  </li>

        
      </ul>
    </nav>
  </li>

    
      
      
      


  <li class="md-nav__item">
    <a href="../changelog/" title="Changelog" class="md-nav__link">
      Changelog
    </a>
  </li>

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              <div class="md-sidebar md-sidebar--secondary" data-md-component="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    
<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
      </span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#representation-of-experimental-data" class="md-nav__link">
    Representation of Experimental Data
  </a>
  
    <nav class="md-nav" aria-label="Representation of Experimental Data">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#subject-terms-perturbations" class="md-nav__link">
    Subject Terms (Perturbations)
  </a>
  
    <nav class="md-nav" aria-label="Subject Terms (Perturbations)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#belv2-how-should-i-represent-chemical-inhibitor-experiments" class="md-nav__link">
    BELv2 How should I represent chemical inhibitor experiments?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-experiments-that-use-site-directed-mutants" class="md-nav__link">
    How do I represent experiments that use site-directed mutants?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#non-phosphorylatable-mutant" class="md-nav__link">
    Non-phosphorylatable mutant
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-observations-resulting-from-manipulation-of-two-or-more-entities" class="md-nav__link">
    How do I represent observations resulting from manipulation of two or more entities?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-should-i-represent-gene-knock-out-or-rnai-experiments" class="md-nav__link">
    How should I represent gene knock out or RNAi experiments?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#gene-knockouts" class="md-nav__link">
    Gene knockouts
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#rna-interference" class="md-nav__link">
    RNA interference
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-should-i-represent-overexpression-experiments" class="md-nav__link">
    How should I represent overexpression experiments?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#when-should-i-use-the-protein-abundance-vs-the-activity-of-a-protein" class="md-nav__link">
    When should I use the protein abundance vs. the activity of a protein?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#effects-are-not-due-to-the-catalytic-activity-of-the-protein" class="md-nav__link">
    Effects are not due to the catalytic activity of the protein
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#effects-are-likely-due-to-the-activity-of-the-protein" class="md-nav__link">
    Effects are likely due to the activity of the protein
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#not-enough-information-is-available" class="md-nav__link">
    Not enough information is available
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#relationships" class="md-nav__link">
    Relationships
  </a>
  
    <nav class="md-nav" aria-label="Relationships">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#when-should-i-use-a-correlative-relationship" class="md-nav__link">
    When should I use a correlative relationship?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#correlative" class="md-nav__link">
    Correlative
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#causal" class="md-nav__link">
    Causal
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#object-terms-measurements" class="md-nav__link">
    Object Terms (Measurements)
  </a>
  
    <nav class="md-nav" aria-label="Object Terms (Measurements)">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#how-should-i-represent-microarray-data" class="md-nav__link">
    How should I represent microarray data?
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#statement-annotations" class="md-nav__link">
    Statement Annotations
  </a>
  
    <nav class="md-nav" aria-label="Statement Annotations">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#how-do-i-annotate-a-relationship-observed-in-multiple-biological-contexts" class="md-nav__link">
    How do I annotate a relationship observed in multiple biological contexts?
  </a>
  
    <nav class="md-nav" aria-label="How do I annotate a relationship observed in multiple biological contexts?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example" class="md-nav__link">
    Example
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#modified-proteins" class="md-nav__link">
    Modified Proteins
  </a>
  
    <nav class="md-nav" aria-label="Modified Proteins">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-a-protein-modification-when-specific-information-is-not-available" class="md-nav__link">
    How do I represent a protein modification when specific information is not available?
  </a>
  
    <nav class="md-nav" aria-label="How do I represent a protein modification when specific information is not available?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example_1" class="md-nav__link">
    Example
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example_2" class="md-nav__link">
    Example
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-a-protein-modification-within-a-complex" class="md-nav__link">
    How do I represent a protein modification within a complex?
  </a>
  
    <nav class="md-nav" aria-label="How do I represent a protein modification within a complex?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#exception-modified-histones-bound-to-the-promoter-of-a-specific-gene" class="md-nav__link">
    Exception – modified histones bound to the promoter of a specific gene
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-a-situation-where-multiple-phosphorylations-are-required-for-a-proteins-activity" class="md-nav__link">
    How do I represent a situation where multiple phosphorylations are required for a protein's activity?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-a-situation-where-one-protein-modification-initiates-additional-modifications" class="md-nav__link">
    How do I represent a situation where one protein modification initiates additional modifications?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-removal-of-a-protein-modification-eg-dephosphorylation-deubiquitination" class="md-nav__link">
    How do I represent removal of a protein modification (e.g., dephosphorylation, deubiquitination)?
  </a>
  
    <nav class="md-nav" aria-label="How do I represent removal of a protein modification (e.g., dephosphorylation, deubiquitination)?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#deubiquitination" class="md-nav__link">
    Deubiquitination
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#dephosphorylation" class="md-nav__link">
    Dephosphorylation
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#reactions" class="md-nav__link">
    Reactions
  </a>
  
    <nav class="md-nav" aria-label="Reactions">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#how-can-i-represent-a-reversible-metabolic-reaction" class="md-nav__link">
    How can I represent a reversible metabolic reaction?
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#when-and-why-should-i-use-a-reaction-term" class="md-nav__link">
    When and why should I use a reaction term?
  </a>
  
    <nav class="md-nav" aria-label="When and why should I use a reaction term?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#when_-should-i-use-a-reaction-term" class="md-nav__link">
    When_ should I use a reaction term
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#why_-should-i-use-a-reaction-term" class="md-nav__link">
    Why_ should I use a reaction term
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#protein-protein-interactions" class="md-nav__link">
    Protein-Protein Interactions
  </a>
  
    <nav class="md-nav" aria-label="Protein-Protein Interactions">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#how-do-i-represent-a-physical-interaction-between-two-entities" class="md-nav__link">
    How do I represent a physical interaction between two entities?
  </a>
  
    <nav class="md-nav" aria-label="How do I represent a physical interaction between two entities?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#examples" class="md-nav__link">
    Examples
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#protein-families" class="md-nav__link">
    Protein Families
  </a>
  
    <nav class="md-nav" aria-label="Protein Families">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#when-should-i-use-a-protein-family-instead-of-a-specific-protein" class="md-nav__link">
    When should I use a protein family instead of a specific protein?
  </a>
  
    <nav class="md-nav" aria-label="When should I use a protein family instead of a specific protein?">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example-1" class="md-nav__link">
    Example 1:
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-2" class="md-nav__link">
    Example 2:
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-3" class="md-nav__link">
    Example 3:
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#more-cookbook" class="md-nav__link">
    More Cookbook
  </a>
  
    <nav class="md-nav" aria-label="More Cookbook">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#causal-statement-examples" class="md-nav__link">
    Causal Statement Examples
  </a>
  
    <nav class="md-nav" aria-label="Causal Statement Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#causal-increase" class="md-nav__link">
    Causal increase
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example_3" class="md-nav__link">
    Example
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#causal-decrease" class="md-nav__link">
    Causal decrease
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example_4" class="md-nav__link">
    Example
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_1" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_1" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#causes-no-change" class="md-nav__link">
    Causes no change
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example_5" class="md-nav__link">
    Example
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_2" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_2" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#correlative-statement-examples" class="md-nav__link">
    Correlative Statement Examples
  </a>
  
    <nav class="md-nav" aria-label="Correlative Statement Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#negativecorrelation" class="md-nav__link">
    negativeCorrelation
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#association" class="md-nav__link">
    association
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_3" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_3" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#direct-causal-statement-examples" class="md-nav__link">
    Direct Causal Statement Examples
  </a>
  
    <nav class="md-nav" aria-label="Direct Causal Statement Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#example-ligand-and-receptor" class="md-nav__link">
    Example - Ligand and Receptor
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_4" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_4" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-kinase-and-substrate" class="md-nav__link">
    Example - Kinase and Substrate
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_5" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-catalyst-and-reaction" class="md-nav__link">
    Example - Catalyst and Reaction
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_6" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_5" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-self-referential-relationships" class="md-nav__link">
    Example - Self-Referential Relationships
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_7" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_6" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#example-direct-transcriptional-control" class="md-nav__link">
    Example - Direct Transcriptional Control
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#long-form_8" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_7" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
          <li class="md-nav__item">
  <a href="#nested-statement-example" class="md-nav__link">
    Nested Statement Example
  </a>
  
    <nav class="md-nav" aria-label="Nested Statement Example">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#long-form_9" class="md-nav__link">
    Long Form
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#short-form_8" class="md-nav__link">
    Short Form
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#other-examples" class="md-nav__link">
    Other Examples
  </a>
  
    <nav class="md-nav" aria-label="Other Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#membership-assignment-examples" class="md-nav__link">
    Membership Assignment Examples
  </a>
  
    <nav class="md-nav" aria-label="Membership Assignment Examples">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#protein-family" class="md-nav__link">
    Protein Family
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#complex-component" class="md-nav__link">
    Complex Component
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-binding-interaction" class="md-nav__link">
    Example - Binding Interaction
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-protein-protein-interaction-as-bel-statement" class="md-nav__link">
    Example - protein-protein interaction as BEL statement
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-protein-protein-interaction-as-statement-object" class="md-nav__link">
    Example - protein-protein interaction as Statement object
  </a>
  
    <nav class="md-nav" aria-label="Example - protein-protein interaction as Statement object">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#protein-dna-interactions" class="md-nav__link">
    Protein – DNA interactions
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-transcription-factor-protein-binding-to-dna" class="md-nav__link">
    Example - transcription factor protein binding to DNA
  </a>
  
    <nav class="md-nav" aria-label="Example - transcription factor protein binding to DNA">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#protein-small-molecule-interactions" class="md-nav__link">
    Protein – small molecule interactions
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#example-protein-binding-to-a-small-molecule" class="md-nav__link">
    Example - protein binding to a small molecule
  </a>
  
    <nav class="md-nav" aria-label="Example - protein binding to a small molecule">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#causal_1" class="md-nav__link">
    Causal
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#causal_2" class="md-nav__link">
    Causal
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content">
            <article class="md-content__inner md-typeset">
              
                
                  <a href="https://github.com/biological-expression-language/biological-expression-language.github.io/edit/master/docs/cookbook.md" title="Edit this page" class="md-content__button md-icon">
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25z"/></svg>
                  </a>
                
                
                  
                
                
                <h1 id="cookbook">Cookbook<a class="headerlink" href="#cookbook" title="Permanent link">&para;</a></h1>
<p>These pages contain suggestions and guidelines for representing scientific findings in BEL.</p>
<ul>
<li>Representation of Experimental Data</li>
<li>Statement Annotations</li>
<li>Modified Proteins</li>
<li>Reactions</li>
<li>Protein-Protein Interactions</li>
<li>Protein Families</li>
</ul>
<h2 id="representation-of-experimental-data">Representation of Experimental Data<a class="headerlink" href="#representation-of-experimental-data" title="Permanent link">&para;</a></h2>
<p>In a causal BEL Statement, the subject term frequently represents an experimentally manipulated entity while the object term represents a measured entity. Our best practices apply different levels of inference for mapping subject and object terms, particularly for representing 'omic data.</p>
<ul>
<li>Subject Terms (Perturbations)</li>
<li>Relationships</li>
<li>Object Terms (Measurements)</li>
</ul>
<h3 id="subject-terms-perturbations">Subject Terms (Perturbations)<a class="headerlink" href="#subject-terms-perturbations" title="Permanent link">&para;</a></h3>
<ul>
<li>BELv2How should I represent chemical inhibitor experiments?</li>
<li>How do I represent experiments that use site-directed mutants?</li>
<li>How do I represent observations resulting from manipulation of two or more entities?</li>
<li>How should I represent gene knock out or RNAi experiments?</li>
<li>How should I represent overexpression experiments?</li>
<li>When should I use the protein abundance vs. the activity of a protein?</li>
</ul>
<h4 id="belv2-how-should-i-represent-chemical-inhibitor-experiments">BELv2 How should I represent chemical inhibitor experiments?<a class="headerlink" href="#belv2-how-should-i-represent-chemical-inhibitor-experiments" title="Permanent link">&para;</a></h4>
<p>For experiments where protein activity is perturbed with a chemical inhibitor, we generally use the chemical as the subject term and not the activity of the target protein. In many cases, the effects of the chemical are not specific to the intended target. This representation approach avoids unintended attribution of off-target effects of a chemical to the target protein.</p>
<p>For example, treatment of cells with the PI3 kinase inhibitor LY294002 significantly decreases expression of TGFB2 RNA (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=20629536[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=20629536[PMID</a> 20629536]):</p>
<div class="highlight"><pre><span></span><code>a(SCHEM:&quot;LY 294002&quot;) -| r(HGNC:TGFB2)
</code></pre></div>

<p>In a case where more information is available, the protein activity targeted by the inhibitor can be used as the subject term. For example, if the effect of LY 29004 on TGFB2 RNA expression was demonstrated to require the PIK3CA gene, we could represent the subject term as the kinase activity of the PIK3CA protein.</p>
<div class="highlight"><pre><span></span><code>act(p(HGNC:PIK3CA), ma(kin)) -&gt; r(HGNC:TGFB2)
</code></pre></div>

<h4 id="how-do-i-represent-experiments-that-use-site-directed-mutants">How do I represent experiments that use site-directed mutants?<a class="headerlink" href="#how-do-i-represent-experiments-that-use-site-directed-mutants" title="Permanent link">&para;</a></h4>
<p>The artificial (laboratory) creation of sequence variants is often used to investigate the effects of protein activity or specific post-translational modifications. These include proteins altered to be constitutively active or dominant negative, as well as proteins with specific amino acid residues altered to prevent phosphorylation. While many of these sequence alterations can be precisely represented with BEL, this may not be the best approach to capturing the observations from experiments that use these constructs.</p>
<h4 id="non-phosphorylatable-mutant">Non-phosphorylatable mutant<a class="headerlink" href="#non-phosphorylatable-mutant" title="Permanent link">&para;</a></h4>
<p>In this example, mutation of FOXO1 serine 256 to alanine is used to block phosphorylation at 256 (S256A), a site phosphorylated by AKT. The S256A mutation was found to impair phosphorylation of threonine 24 and serine 319 by AKT (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11237865[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=11237865[PMID</a> 11237865]). We could represent this observation as follows:</p>
<div class="highlight"><pre><span></span><code>p(HGNC:FOXO1, var(&quot;p.Ser256Ala&quot;)) =| p(HGNC:FOXO1, pmod(Ph, Ser, 256))
</code></pre></div>

<div class="highlight"><pre><span></span><code>p(HGNC:FOXO1, var(&quot;p.Ser256Ala&quot;)) =| (p(SFAM:&quot;AKT Family&quot;) =&gt; p(HGNC:FOXO1, pmod(Ph, Thr, 24)))
</code></pre></div>

<div class="highlight"><pre><span></span><code>p(HGNC:FOXO1, var(&quot;p.Ser256Ala&quot;)) =| (p(SFAM:&quot;AKT Family&quot;) =&gt; p(HGNC:FOXO1, pmod(Ph, Ser, 319)))
</code></pre></div>

<p>The first statement indicates that phosphorylation at S256 is blocked by mutation of S256 to alanine. The next two statements indicate that the S256A mutation decreases phosphorylation of FOXO1 threonine 24 and serine 319 by AKT. However, we are not generally interested in the effects of a lab-created mutant like S256A so much as the role of phosphorylation at serine 256 on phosphorylation of the other two sites. Thus, we recommend the following representation:</p>
<div class="highlight"><pre><span></span><code>p(HGNC:FOXO1, pmod(Ph, Ser, 256)) =&gt; (p(SFAM:&quot;AKT Family&quot;) =&gt; p(HGNC:FOXO1, pmod(Ph, Thr, 24)))
p(HGNC:FOXO1, pmod(Ph, Ser, 256)) =&gt; (p(SFAM:&quot;AKT Family&quot;) =&gt; p(HGNC:FOXO1, pmod(Ph, Ser, 319)))
</code></pre></div>

<p>Here, the statements indicate that phosphorylation of FOXO1 at S256 increases the phosphorylation of T24 and S319 by the kinase activity of AKT. While both representations are accurate, the second version is better suited to integrating other information about the role of FOXO1 phosphorylation at S256 into a cohesive, traversable model.</p>
<h4 id="how-do-i-represent-observations-resulting-from-manipulation-of-two-or-more-entities">How do I represent observations resulting from manipulation of two or more entities?<a class="headerlink" href="#how-do-i-represent-observations-resulting-from-manipulation-of-two-or-more-entities" title="Permanent link">&para;</a></h4>
<p>In some cases an experiment has a complex perturbation, where manipulations of multiple biological entities are required for an effect. Multiple BEL abundance terms can be represented together as the subject of a BEL Statement by using the <code>compositeAbundance()</code> or <code>composite()</code> function.</p>
<p>In this example, TGF-beta cooperates with IL-6 to generate T-helper 17 cells (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17918200[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=17918200[PMID</a> 17918200]):</p>
<div class="highlight"><pre><span></span><code>composite(p(MGI:Tgfb1), p(MGI:Il6)) -&gt; bp(GO:&quot;T-helper 17 cell differentiation&quot;)
</code></pre></div>

<p>If the two manipulated components are known to physically interact (such as a receptor and it's ligand), we recommend inferring their effects rather than using a composite term.</p>
<p>In this example, both Met and Hgf (the Met ligand) are required for increased expression of integrin Itgav RNA (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=16710476[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=16710476[PMID</a> 16710476]):</p>
<p>Not recommended:</p>
<div class="highlight"><pre><span></span><code> composite(p(MGI:Hgf), p(MGI:Met)) -&gt; r(MGI:Itgav)
</code></pre></div>

<p>Recommended:</p>
<p>kin(p(MGI:Met)) -&gt; r(MGI:Itgav)</p>
<p>p(MGI:Hgf) -&gt; r(MGI:Itgav)</p>
<p>Because Hgf binds to and directly activates Met, the effect of Met and Hgf together on Itgav RNA expression can be inferred to result from Met activity.</p>
<h4 id="how-should-i-represent-gene-knock-out-or-rnai-experiments">How should I represent gene knock out or RNAi experiments?<a class="headerlink" href="#how-should-i-represent-gene-knock-out-or-rnai-experiments" title="Permanent link">&para;</a></h4>
<p>Our general practice is to represent the subject term for experiments where the perturbation is a gene deletion or RNAi knockdown as the abundance of the corresponding protein.</p>
<h4 id="gene-knockouts">Gene knockouts<a class="headerlink" href="#gene-knockouts" title="Permanent link">&para;</a></h4>
<p>In this example, mice with a gene deletion of Nfe2l2 express reduced mRNA of the glutathione S transferase Gsta1 compared to wild-type mice (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=11991805[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=11991805[PMID</a> 11991805]):</p>
<p>p(MGI:Nfe2l2) -&gt; r(MGI:Gsta1)</p>
<h4 id="rna-interference">RNA interference<a class="headerlink" href="#rna-interference" title="Permanent link">&para;</a></h4>
<p>In this example, knockdown of PTEN using RNA interference results in increased CDKN1A protein levels (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17300726[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=17300726[PMID</a> 17300726]):</p>
<p>p(HGNC:PTEN) -| p(HGNC:CDKN1A)</p>
<p>We assume that the effects of PTEN RNAi are due to knock down of PTEN protein. Decreased PTEN protein resulting in increased CDKN1A protein is interpreted as PTEN decreases CDKN1A protein.</p>
<p>It is generally preferable to represent the subject term as the protein abundance and not an activity of the protein, particularly for 'omic experiments. See also When should I use the protein abundance vs. the activity of a protein?</p>
<h4 id="how-should-i-represent-overexpression-experiments">How should I represent overexpression experiments?<a class="headerlink" href="#how-should-i-represent-overexpression-experiments" title="Permanent link">&para;</a></h4>
<p>For experiments where the perturbation is overexpression of DNA or RNA for the purpose of overexpressing a protein, we generally represent the subject term as a protein abundance.</p>
<p>In this example, SIAH2 and repp86 (TPX2) proteins interact, and overexpression of SIAH2 by transfection increases degradation of TPX2 protein (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17716627[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=17716627[PMID</a> 17716627]):</p>
<div class="highlight"><pre><span></span><code>p(HGNC:SIAH2) =&gt; deg(p(HGNC:TPX2))
</code></pre></div>

<p>The statement is modeled as direct because the subject and object term proteins interact.</p>
<p>While it would be technically correct to represent overexpressions achieved via DNA transfection as gene abundances and those from mRNA transfections as RNA abundances, this distinction is not useful for applications like Whistle and pathfinding. It is generally preferable to represent the subject term as the protein abundance and not an activity of the protein, particularly for 'omic experiments, if it is not clear that the activity is required or responsible for the effect.</p>
<h4 id="when-should-i-use-the-protein-abundance-vs-the-activity-of-a-protein">When should I use the protein abundance vs. the activity of a protein?<a class="headerlink" href="#when-should-i-use-the-protein-abundance-vs-the-activity-of-a-protein" title="Permanent link">&para;</a></h4>
<p>Many experimental perturbations involve the overexpression or knockdown of protein abundance. We generally represent this type of experiment using a protein abundance as the subject term instead of an activity (e.g., kinase or phosphatase) of the protein. While many proteins have a known activity, this activity is not always responsible for all downstream effects of overexpression or knockdown; the same effects occur when either wild-type or catalytically inactive forms are expressed.</p>
<p>Below are examples of BEL statements for cases where:</p>
<p>. the effects of increased or decreased protein abundance are not due to the catalytic activity of the protein, . effects are likely due to an increase or decrease in the activity of the protein, and . not enough information is available.</p>
<h4 id="effects-are-not-due-to-the-catalytic-activity-of-the-protein">Effects are not due to the catalytic activity of the protein<a class="headerlink" href="#effects-are-not-due-to-the-catalytic-activity-of-the-protein" title="Permanent link">&para;</a></h4>
<p><em>Example 1.</em> A mutant telomerase (TERT) protein lacking telomerase activity retains its effects on keratinocyte proliferation (<a href="http://www.ncbi.nlm.nih.gov/pubmed/18208333[PMID">http://www.ncbi.nlm.nih.gov/pubmed/18208333[PMID</a> 18208333]):</p>
<div class="highlight"><pre><span></span><code>p(MGI:Tert) -&gt; bp(GO:&quot;cell proliferation&quot;)
act(p(MGI:Tert), ma(cat)) causesNoChange bp(GO:&quot;cell proliferation&quot;)
</code></pre></div>

<p><em>Example 2.</em> The serine-threonine kinase RIPK1 activates NF-kappaB through a mechanism that does not involve the protein's kinase activity (<a href="http://www.ncbi.nlm.nih.gov/pubmed/20354226[PMID">http://www.ncbi.nlm.nih.gov/pubmed/20354226[PMID</a> 20354226]).</p>
<div class="highlight"><pre><span></span><code>p(HGNC:RIPK1) -&gt; act(p(GOCC:&quot;NF-kappaB complex&quot;), ma(tscript))
act(p(HGNC:RIPK1), ma(kin)) causesNoChange act(p(GOCC:&quot;NF-kappaB complex&quot;), ma(tscript))
</code></pre></div>

<h4 id="effects-are-likely-due-to-the-activity-of-the-protein">Effects are likely due to the activity of the protein<a class="headerlink" href="#effects-are-likely-due-to-the-activity-of-the-protein" title="Permanent link">&para;</a></h4>
<p><em>Example.</em> Genes are differentially expressed in wild-type vs. Met knock out mouse hepatocytes, only after treatment with Hgf, the Met ligand (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16710476[PMID">http://www.ncbi.nlm.nih.gov/pubmed/16710476[PMID</a> 16710476]). These genes include integrins Itgav, Itga3, and Itgb1.</p>
<div class="highlight"><pre><span></span><code>act(p(MGI:Met), ma(kin)) -&gt; r(MGI:Itgav)
act(p(MGI:Met), ma(kin)) -&gt; r(MGI:Itga3)
act(p(MGI:Met), ma(kin)) -&gt; r(MGI:Itgb1)
</code></pre></div>

<p>Because both Met and the Met ligand are required for the increase in gene expression, we use the activity of Met as the subject term.</p>
<h4 id="not-enough-information-is-available">Not enough information is available<a class="headerlink" href="#not-enough-information-is-available" title="Permanent link">&para;</a></h4>
<p><em>Example.</em> Genes are differentially expressed in the pancreas of mice with a pancreas-specific beta-catenin deletion (<a href="http://www.ncbi.nlm.nih.gov/pubmed/17222338[PMID">http://www.ncbi.nlm.nih.gov/pubmed/17222338[PMID</a> 17222338]). These include decreased expression of the hedgehog interacting protein Hhip in the knock-out compared to wild-type.</p>
<div class="highlight"><pre><span></span><code>p(MGI:Ctnnb1) -&gt; r(MGI:Hhip)
</code></pre></div>

<p>In this case, not enough information is available to determine if the change in Hhip expression is due to Ctnnb1 function in transcription, cell adhesion, or another role.</p>
<h3 id="relationships">Relationships<a class="headerlink" href="#relationships" title="Permanent link">&para;</a></h3>
<h4 id="when-should-i-use-a-correlative-relationship">When should I use a correlative relationship?<a class="headerlink" href="#when-should-i-use-a-correlative-relationship" title="Permanent link">&para;</a></h4>
<p>Correlative Relationships, Correlative relationships are more appropriate than Causal Relationships, causal relationships to represent observations that do not clearly result from an experimental perturbation. BEL causal relationships include <code>increases</code>, <code>decreases</code>, <code>directlyIncreases</code>, <code>directlyDecreases</code>, and <code>regulates</code>. Correlative relationships include <code>positiveCorrelation</code> and <code>negativeCorrelation</code>.</p>
<h4 id="correlative">Correlative<a class="headerlink" href="#correlative" title="Permanent link">&para;</a></h4>
<p>If the observation comes from the comparison of human tumors grouped by the occurrence of a specific mutation, then the relationship should generally be expressed as correlative. In this case there is no experimental perturbation. In this example, most patient tumor samples with an EGFR L858R mutation were observed to exhibit a reduction in ERBB2 tyrosine 1248 phosphorylation compared to wild-type samples (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18687633[PMID">http://www.ncbi.nlm.nih.gov/pubmed/?term=18687633[PMID</a> 18687633]):</p>
<div class="highlight"><pre><span></span><code>p(HGNC:EGFR, var(&quot;p.Leu858Arg&quot;)) negativeCorrelation p(HGNC:ERBB2, pmod(Ph,Tyr,1248))
</code></pre></div>

<p>In this case, no evidence is presented to suggest that the differences in ERBB2 phosphorylation are causally related to the EGFR mutation, only that the two observations are inversely correlated. Note that the subject and object terms are interchangeable for correlative relationships.</p>
<h4 id="causal">Causal<a class="headerlink" href="#causal" title="Permanent link">&para;</a></h4>
<p>If the observation comes from the comparison of experimentally controlled states, like gene deletion, overexpression, or introduction of a mutant allele into a cell line or animal, the experimental perturbation can generally be represented as the subject term of a causal statement. In this example, DUSP6 RNA is observed to be upregulated in immortalized human bronchial epithelial cells transfected with EGFR mutant L858R, as compared to WT EGFR (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16489012[PMID">http://www.ncbi.nlm.nih.gov/pubmed/16489012[PMID</a> 16489012]):</p>
<div class="highlight"><pre><span></span><code>p(HGNC:EGFR, var(&quot;p.Leu858Arg&quot;)) -&gt; r(HGNC:DUSP6)
</code></pre></div>

<p>In this case, the EGFR mutation is introduced as an experimentally-controlled perturbation.</p>
<h3 id="object-terms-measurements">Object Terms (Measurements)<a class="headerlink" href="#object-terms-measurements" title="Permanent link">&para;</a></h3>
<h4 id="how-should-i-represent-microarray-data">How should I represent microarray data?<a class="headerlink" href="#how-should-i-represent-microarray-data" title="Permanent link">&para;</a></h4>
<p>We record the results of experiments like microarrays and RT-PCR, which measure RNA abundances, by representing the object terms as RNA abundances. Only significant effects (e.g., meeting minimum criteria for fold change and statistical significance) should be recorded in BEL Statements.</p>
<p>In a causal BEL Statement, the subject term generally represents an experimentally manipulated entity while the object term represents a measured entity. Our general practice is to represent the object terms in BEL Statements with the terms most closely related to the experimental measurement.</p>
<p>This direct representation of the measurement in BEL supports the creation of KAMs to which 'omic data can be mapped directly and analyzed using automated reasoning applications like Whistle. Inference of the potential downstream consequences of RNA expression changes is supported by connection of RNA abundances to the corresponding proteins during knowledge network compilation</p>
<h2 id="statement-annotations">Statement Annotations<a class="headerlink" href="#statement-annotations" title="Permanent link">&para;</a></h2>
<h3 id="how-do-i-annotate-a-relationship-observed-in-multiple-biological-contexts">How do I annotate a relationship observed in multiple biological contexts?<a class="headerlink" href="#how-do-i-annotate-a-relationship-observed-in-multiple-biological-contexts" title="Permanent link">&para;</a></h3>
<p>Often, the scientific literature reports a relationship as occurring across several biological contexts.</p>
<p>Our general practice is to represent each observation with a separate statement. Several annotations can be used to describe the same context, e.g., 'lung' and 'fibroblast', but distinct BEL statements should be used to describe each experimental context that the relationship is observed in.</p>
<h4 id="example">Example<a class="headerlink" href="#example" title="Permanent link">&para;</a></h4>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/18650932[PMID">http://www.ncbi.nlm.nih.gov/pubmed/18650932[PMID</a> 18650932] - siRNA knockdown of the atypical PKC-interacting protein Par-4 (PAWR) increases phosphorlyation of AKT at Serine 473 in both human 293 and A549 cells.</p>
<p>"To test whether this is also true in human cells, we used a Par-4 siRNA to deplete endogenous Par-4 levels in human 293 cells and in the A549 human lung adenocarcinoma cell line. Cells were treated with control or Par-4-specific siRNAs, after which they were kept for 24 h in serum-free medium conditions and then stimulated with serum. Data in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519103/figure/f5/[Figure">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519103/figure/f5/[Figure</a> 5E and F] clearly demonstrate that the knockdown of Par-4 provokes enhanced serum-activated phospho-Akt-Ser473 levels in A549 and 293 human cells, respectively."</p>
<p>Not recommended:</p>
<div class="highlight"><pre><span></span><code>SET CellLine = {A549, 293}
p(HGNC:PAWR) -| p(SFAM:&quot;AKT Family&quot;, pmod(Ph, Ser, 473))
</code></pre></div>

<p>Recommended:</p>
<div class="highlight"><pre><span></span><code>SET CellLine = A549

p(HGNC:PAWR) -| p(SFAM:&quot;AKT Family&quot;, pmod(Ph, Ser, 473))

SET CellLine

p(HGNC:PAWR) -| p(SFAM:&quot;AKT Family&quot;, pmod(Ph, Ser, 473))
</code></pre></div>

<h2 id="modified-proteins">Modified Proteins<a class="headerlink" href="#modified-proteins" title="Permanent link">&para;</a></h2>
<ul>
<li>How do I represent a protein modification when specific information is not available?</li>
<li>How do I represent a protein modification within a complex?</li>
<li>Xmultphosphforprotein, How do I represent a situation where multiple phosphorylations are required for a protein's activity?</li>
<li>How do I represent a situation where one protein modification initiates additional modifications?</li>
<li>Xremovalofproteinmod, How do I represent removal of a protein modification (e.g., dephosphorylation, deubiquitination)?</li>
</ul>
<h3 id="how-do-i-represent-a-protein-modification-when-specific-information-is-not-available">How do I represent a protein modification when specific information is not available?<a class="headerlink" href="#how-do-i-represent-a-protein-modification-when-specific-information-is-not-available" title="Permanent link">&para;</a></h3>
<p>BEL terms for Protein Modifications, post-translational modifications of proteins specify the type of modification, the modified amino acid, and the position of the modified amino acid. The modified amino acid and position are not required, so protein modifications can be represented with less specific information.</p>
<h4 id="example_1">Example<a class="headerlink" href="#example_1" title="Permanent link">&para;</a></h4>
<p>Human AKT1 protein modified by phosphorylation at serine 473</p>
<p>p(HGNC:AKT1, pmod(Ph, Ser, 473))</p>
<p>Human AKT1 protein modified by phosphorylation at an unspecified serine residue</p>
<p>p(HGNC:AKT1, pmod(Ph, Ser))</p>
<p>Human AKT1 protein that has been modified by phosphorylation at an unspecified amino acid residue</p>
<p>p(HGNC:AKT1, pmod(Ph))</p>
<p>As a general rule, if specific information is available, it should be used. In some cases, this involves investigation sections of a paper outside of the evidence text or other referenced papers to determine which specific modifications have been measured.</p>
<p>Non-specific protein modification terms have limited value in the context of a knowledge network. For example, phosphorylation at different sites of the same protein can have opposing effects. For example: "Akt-phosphorylated FOXO interacts with the ubiquitin ligase Skp2 and is targeted for proteasomal degradation" (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15917664[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=15917664[PMID</a> 15917664])</p>
<h4 id="example_2">Example<a class="headerlink" href="#example_2" title="Permanent link">&para;</a></h4>
<p>Recommended:</p>
<p>act(p(SFAM:"AKT Family"), ma(kin)) =&gt; (act(p(HGNC:SKP2)) =&gt; deg(p(SFAM:"FOXO Family")))</p>
<p>Not recommended:</p>
<div class="highlight"><pre><span></span><code>p(SFAM:&quot;FOXO Family&quot;, pmod(Ph)) =&gt; (act(p(HGNC:SKP2)) =&gt; deg(p(SFAM:&quot;FOXO Family&quot;)))
</code></pre></div>

<p>The first BEL Statement indicates that the kinase activity of AKT increases the degradation of FOXO by SKP2. The second statement indicates that phosphorylation of FOXO increases the degradation of FOXO by SKP2. In this case, more information is captured by using the phosphorylating kinase AKT as the subject term instead of the non-specified phosphorylation of FOXO.</p>
<h3 id="how-do-i-represent-a-protein-modification-within-a-complex">How do I represent a protein modification within a complex?<a class="headerlink" href="#how-do-i-represent-a-protein-modification-within-a-complex" title="Permanent link">&para;</a></h3>
<p>In many cases, complexes include proteins with post-translational modifications and these modifications influence complex formation. For example, HIF1A that has been hydroxylated on proline residues 402 and 564 interacts with VHL (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17925579[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=17925579[PMID</a> 17925579]).</p>
<p>Our general practice is to represent this type of event as a causal statement in BEL, with the modified protein as the subject term and the complex with no specified modifications as the object term. Because the modified protein is a component of the complex, we use a direct causal relationship:</p>
<div class="highlight"><pre><span></span><code>p(HGNC:HIF1A, pmod(Hy, Pro, 402)) =&gt; complex(p(HGNC:HIF1A), p(HGNC:VHL))

p(HGNC:HIF1A, pmod(Hy, Pro, 564)) =&gt; complex(p(HGNC:HIF1A), p(HGNC:VHL))
</code></pre></div>

<p>While BEL allows representation of complexes with the modified proteins as components, we do not recommend this approach:</p>
<div class="highlight"><pre><span></span><code>complex(p(HGNC:HIF1A, pmod(Hy, Pro, 402)), p(HGNC:VHL))
</code></pre></div>

<p>The practice of composing a complex using protein abundances without any specified modifications provides a standardized representation for complexes and allows the effects of modifications on complex formation to be captured as causal relationships. The modified forms of HIF1A, <code>p(HGNC:HIF1A, pmod(Hy, Pro, 402))</code> and <code>p(HGNC:HIF1A, pmod(Hy, Pro, 564))</code> are considered a subset of the total <code>p(HGNC:HIF1A)</code>.</p>
<p>This approach enables representation of the effects of multiple protein modifications on complex formation by using a causal statement for each modification.</p>
<p>BELv2.0 does not provide a specific representation of unmodified protein abundances.</p>
<h4 id="exception-modified-histones-bound-to-the-promoter-of-a-specific-gene"><em>Exception – modified histones bound to the promoter of a specific gene</em><a class="headerlink" href="#exception-modified-histones-bound-to-the-promoter-of-a-specific-gene" title="Permanent link">&para;</a></h4>
<p>One exception to our general practice of not specifying protein modifications within complex abundances is the interaction of specific modified histones with the promoter of a specific gene. In this example, cigarette smoke is observed to increase H3K27me3 levels at the DKK1 promoter (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19351856[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=19351856[PMID</a> 19351856]).</p>
<div class="highlight"><pre><span></span><code> a(SCHEM:&quot;smoke condensate, cigarette (gas phase)&quot;) -&gt; \
    complex(p(PFH:&quot;Histone H3 Family&quot;,pmod(Me3, Lys, 27)), g(HGNC:DKK1))
</code></pre></div>

<p>In this example, the modification of the histone by trimethylation does not affect its binding to the gene DKK1. In addition, cigarette smoke does not increase or decrease the overall abundance of the modified histone, only the abundance of the modified histone at the DKK1 promoter.</p>
<h3 id="how-do-i-represent-a-situation-where-multiple-phosphorylations-are-required-for-a-proteins-activity">How do I represent a situation where multiple phosphorylations are required for a protein's activity?<a class="headerlink" href="#how-do-i-represent-a-situation-where-multiple-phosphorylations-are-required-for-a-proteins-activity" title="Permanent link">&para;</a></h3>
<p>In many cases two or more distinct modifications are required simultaneously for protein activity, and neither modification alone is sufficient. For example, MAPK3 must be phosphorylated at two sites, Threonine 202 and Tyrosine 204, to be active. Our general practice is to take the simple, most general approach, and model the effect of each site separately:</p>
<div class="highlight"><pre><span></span><code>p(HGNC:MAPK3, pmod(Ph, Thr, 202)) =&gt; act(p(HGNC:MAPK3), ma(kin))

p(HGNC:MAPK3, pmod(Ph, Tyr, 204)) =&gt; act(p(HGNC:MAPK3), ma(kin))
</code></pre></div>

<p>A multiply-modified abundance term can be used if it is of high importance to capture the requirement for both modifications:</p>
<div class="highlight"><pre><span></span><code>p(HGNC:MAPK3, pmod(Ph, Thr, 202)), pmod(Ph, Tyr, 204)) =&gt; act(p(HGNC:MAPK3), ma(kin))
</code></pre></div>

<h3 id="how-do-i-represent-a-situation-where-one-protein-modification-initiates-additional-modifications">How do I represent a situation where one protein modification initiates additional modifications?<a class="headerlink" href="#how-do-i-represent-a-situation-where-one-protein-modification-initiates-additional-modifications" title="Permanent link">&para;</a></h3>
<p>In many cases a specific protein modification may be dependent on another modification of the same protein. In this case, the first protein modification can be modeled as the upstream cause of the second. In this example, phosphorylation of CTNNB1 at Serine 45 initiates phosphorylation of CTNNB1 at other sites including Threonine 41 by GSK3 (<a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16618120[PMID">http://www.ncbi.nlm.nih.gov/pubmed/?term=16618120[PMID</a> 16618120]):</p>
<div class="highlight"><pre><span></span><code>p(HGNC:CTNNB1, pmod(Ph, Ser, 45)) =&gt; p(HGNC:CTNNB1, pmod(Ph, Thr, 41))
</code></pre></div>

<p>We represent this relationship as direct, because the subject and object terms have the same root abundance node.</p>
<p>Because the kinase mediating the second phosphorylation is known, this relationship can be modeled alternatively as a nested statement:</p>
<div class="highlight"><pre><span></span><code>p(HGNC:CTNNB1, pmod(Ph, Ser, 45)) =&gt; \
    (kin(p(SFAM:&quot;GSK3 Family&quot;)) =&gt; p(HGNC:CTNNB1, pmod(Ph, Thr, 41)))
</code></pre></div>

<h3 id="how-do-i-represent-removal-of-a-protein-modification-eg-dephosphorylation-deubiquitination">How do I represent removal of a protein modification (e.g., dephosphorylation, deubiquitination)?<a class="headerlink" href="#how-do-i-represent-removal-of-a-protein-modification-eg-dephosphorylation-deubiquitination" title="Permanent link">&para;</a></h3>
<p>Removal of a specific protein modification is represented simply as a decrease in the abundance of the modified protein.</p>
<h4 id="deubiquitination">Deubiquitination<a class="headerlink" href="#deubiquitination" title="Permanent link">&para;</a></h4>
<p>In this example, STAMBP deubiquitinates F2RL1 protein (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=19684015[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=19684015[PMID</a> 19684015]):</p>
<div class="highlight"><pre><span></span><code>act(p(HGNC:STAMBP)) =| p(HGNC:F2RL1, pmod(Ub))
</code></pre></div>

<p>Deubiquitination is represented simply as a decrease in the ubiquitinated form of the protein. Because in this example STAMBP is the deubiquitinating enzyme, we used a directlyDecreases relationship.</p>
<h4 id="dephosphorylation">Dephosphorylation<a class="headerlink" href="#dephosphorylation" title="Permanent link">&para;</a></h4>
<p>In this example, the phosphatase CDC25C dephosphorylates CDK1 at tyrosine 15 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=1384126[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=1384126[PMID</a> 1384126]):</p>
<div class="highlight"><pre><span></span><code>act(p(HGNC:CDC25C), ma(phos)) =| p(HGNC:CDK1, pmod(Ph, Tyr, 15))
</code></pre></div>

<p>Similar to deubiquitination, dephosphorylation is represented simply as a decrease in the modified form of the protein.</p>
<h2 id="reactions">Reactions<a class="headerlink" href="#reactions" title="Permanent link">&para;</a></h2>
<ul>
<li>How can I represent a reversible metabolic reaction?</li>
<li>When and why should I use a reaction term?</li>
</ul>
<h3 id="how-can-i-represent-a-reversible-metabolic-reaction">How can I represent a reversible metabolic reaction?<a class="headerlink" href="#how-can-i-represent-a-reversible-metabolic-reaction" title="Permanent link">&para;</a></h3>
<p>A reversible reaction can be represented by modeling the reaction with the products and reactants interchanged.</p>
<p>For example, HSD11B1 acts primarily to convert cortisone to active cortisol, but in some cell types the reverse reaction is favored (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=12530648[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=12530648[PMID</a> 12530648]):</p>
<div class="highlight"><pre><span></span><code> act(p(HGNC:HSD11B1), ma(cat)) =&gt; \
  rxn(reactants(a(CHEBI:NADPH), a(CHEBI:cortisone)), products(a(CHEBI:&quot;NADP(+)&quot;), a(CHEBI:cortisol)))

 act(p(HGNC:HSD11B1), ma(cat)) =&gt; \
  rxn(reactants(a(CHEBI:&quot;NADP(+)&quot;), a(CHEBI:cortisol)), products(a(CHEBI:NADPH), a(CHEBI:cortisone)))
</code></pre></div>

<p>The top statement represents the forward reaction and the bottom statement represents the reverse reaction.</p>
<h3 id="when-and-why-should-i-use-a-reaction-term">When and why should I use a reaction term?<a class="headerlink" href="#when-and-why-should-i-use-a-reaction-term" title="Permanent link">&para;</a></h3>
<h4 id="when_-should-i-use-a-reaction-term">When_ should I use a reaction term<a class="headerlink" href="#when_-should-i-use-a-reaction-term" title="Permanent link">&para;</a></h4>
<p>Reaction terms allow the representation of a transformation of a list of reactants into a list of products. In this example, the superoxide dismutase SOD1 converts superoxide to hydrogen peroxide:</p>
<div class="highlight"><pre><span></span><code>act(p(HGNC:SOD1), ma(cat)) =&gt; rxn(reactants(a(CHEBI:superoxide)), products(a(CHEBI:&quot;hydrogen peroxide&quot;)))
</code></pre></div>

<p>It is not necessary to include all reactants and products, especially if they are ubiquitous small molecules. In the above example the reactant hydrogen and product oxygen have been omitted from the reaction.</p>
<h4 id="why_-should-i-use-a-reaction-term">Why_ should I use a reaction term<a class="headerlink" href="#why_-should-i-use-a-reaction-term" title="Permanent link">&para;</a></h4>
<p>It is possible to represent the above reaction with separate statements linking the activity of SOD1 to decreased abundances of the reactants and increased abundances of the products:</p>
<div class="highlight"><pre><span></span><code>act(p(HGNC:SOD1), ma(cat)) =| a(CHEBI:superoxide)
act(p(HGNC:SOD1), ma(cat)) =&gt; a(CHEBI:&quot;hydrogen peroxide&quot;)
</code></pre></div>

<p>While this representation describes the function of the catalytic enzyme SOD1, it does not link the product hydrogen peroxide to the reactant superoxide.</p>
<h2 id="protein-protein-interactions">Protein-Protein Interactions<a class="headerlink" href="#protein-protein-interactions" title="Permanent link">&para;</a></h2>
<h3 id="how-do-i-represent-a-physical-interaction-between-two-entities">How do I represent a physical interaction between two entities?<a class="headerlink" href="#how-do-i-represent-a-physical-interaction-between-two-entities" title="Permanent link">&para;</a></h3>
<p>You can use a complex abundance to represent binding events between two or more abundance terms. The subject term of your BEL Statement will be the complex. BEL Statements do not require a relationship and object term.</p>
<p>The order in which the members of the complex are listed is not important.</p>
<h4 id="examples">Examples<a class="headerlink" href="#examples" title="Permanent link">&para;</a></h4>
<p>EPOR physically interacts with CSF2RB, the common beta-receptor (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15456912[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=15456912[PMID</a> 15456912])</p>
<div class="highlight"><pre><span></span><code> complex(p(MGI:EPOR), p(MGI:CSF2RB))
</code></pre></div>

<p>KEAP1 binds 15-deoxy-Delta(12,14)-prostaglandin J2 (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15917255[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=15917255[PMID</a> 15917255])</p>
<div class="highlight"><pre><span></span><code> complex(p(HGNC:KEAP1), a(CHEBI:&quot;15-deoxy-Delta(12,14)-prostaglandin J2&quot;))
</code></pre></div>

<p>KEAP1, CUL3, and RBX1 copurify and are part of a functional E3 ubiquitin ligase complex (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=15572695[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=15572695[PMID</a> 15572695])</p>
<div class="highlight"><pre><span></span><code> complex(p(HGNC:KEAP1), p(HGNC:RBX1), p(HGNC:CUL3))
</code></pre></div>

<p>The AP-1 transcription complex binds the CCL23 promoter (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=17368823[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=17368823[PMID</a> 17368823])</p>
<div class="highlight"><pre><span></span><code> complex(p(GOCC:&quot;AP1 complex&quot;), g(HGNC:CCL23))
</code></pre></div>

<h2 id="protein-families">Protein Families<a class="headerlink" href="#protein-families" title="Permanent link">&para;</a></h2>
<h3 id="when-should-i-use-a-protein-family-instead-of-a-specific-protein">When should I use a protein family instead of a specific protein?<a class="headerlink" href="#when-should-i-use-a-protein-family-instead-of-a-specific-protein" title="Permanent link">&para;</a></h3>
<p>Protein families can be used to represent protein abundances in cases where the information presented by the source does not allow identification of the specific protein. For example:</p>
<h4 id="example-1">Example 1:<a class="headerlink" href="#example-1" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code>SET Citation = &quot;pubmed:11715018&quot;
SET Evidence = &quot;Akt physically associates with MDM2 and phosphorylates it at Ser166 and Ser186.&quot;

act(p(fplx:AKT), ma(kin)) =&gt; p(HGNC:MDM2, pmod(Ph,Ser,166))
act(p(fplx:AKT), ma(kin)) =&gt; p(HGNC:MDM2, pmod(Ph,Ser,186))
</code></pre></div>

<p>Here, Akt may refer to AKT1, AKT2, and/or AKT3.</p>
<h4 id="example-2">Example 2:<a class="headerlink" href="#example-2" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code>SET Citation = &quot;pubmed:17003045&quot;
SET Evidence = &quot;We show that Siah2 is subject to phosphorylation by p38 MAPK ... Phosphopeptide mapping identified T24 and S29 as the primary phospho-acceptor sites.&quot;

act(p(PFH:&quot;MAPK p38 Family&quot;), ma(kin)) =&gt; p(HGNC:SIAH2, pmod(Ph, Thr, 24))
act(p(PFH:&quot;MAPK p38 Family&quot;), ma(kin)) =&gt; p(HGNC:SIAH2, pmod(Ph, Ser, 29))
</code></pre></div>

<p>Here, it is not clear which specific p38 MAPK is responsible (MAPK11, MAPK12, MAPK13, or MAPK14).</p>
<h4 id="example-3">Example 3:<a class="headerlink" href="#example-3" title="Permanent link">&para;</a></h4>
<p>"Hip encodes a membrane glycoprotein that binds to all three mammalian Hedgehog proteins." (<a href="http://www.ncbi.nlm.nih.gov/pubmed?term=10050855[PMID">http://www.ncbi.nlm.nih.gov/pubmed?term=10050855[PMID</a> 10050855])</p>
<p>complex(p(SFAM:"Hedgehog Family"), p(MGI:Hhip))</p>
<p>complex(p(MGI:Ihh), p(MGI:Hhip))</p>
<p>complex(p(MGI:Shh), p(MGI:Hhip))</p>
<p>complex(p(MGI:Dhh), p(MGI:Hhip))</p>
<p>In this case, all three hedgehog family members are reported to bind to the hedgehog interacting protein (Hhip). Statements can be modeled using the family as well as each individual member.</p>
<h2 id="more-cookbook">More Cookbook<a class="headerlink" href="#more-cookbook" title="Permanent link">&para;</a></h2>
<h3 id="causal-statement-examples">Causal Statement Examples<a class="headerlink" href="#causal-statement-examples" title="Permanent link">&para;</a></h3>
<p>Causal statements connect subject and object terms with a causal <code>&lt;&lt;Xincreases, increases&gt;&gt;</code>, <code>&lt;&lt;Xdecreases, decreases&gt;&gt;</code>, or <code>&lt;&lt;Xcnc, causesNoChange&gt;&gt;</code> relationship. Subject terms can be an abundance or process (including activities and transformations) and object terms can be either an abundance, a process, or a second BEL Statement.</p>
<ul>
<li>&lt;&gt;</li>
<li>&lt;&gt;</li>
<li>&lt;&gt;</li>
</ul>
<h4 id="causal-increase">Causal increase<a class="headerlink" href="#causal-increase" title="Permanent link">&para;</a></h4>
<h4 id="example_3">Example<a class="headerlink" href="#example_3" title="Permanent link">&para;</a></h4>
<p>These statements use the causal <code>&lt;&lt;Xincreases, increases&gt;&gt;</code> relationship. These statements are annotated with a citation and supporting evidence text, as well as with the cell line and species context for the experimental observations represented by the statements. These two statements represent the observation that increases in IL6 protein abundance cause increases in the RNA abundance of ENO1 and XBP1. These statements are annotated with CellLine and Species to indicate that the experimental observation was made in the context of the cell line "U266" and species "9606" (Homo sapiens).</p>
<h4 id="long-form">Long Form<a class="headerlink" href="#long-form" title="Permanent link">&para;</a></h4>
<p>SET Citation = {"PubMed", "Int J Oncol 1999 Jul 15(1) 173-8", "10375612"}</p>
<p>SET Support = "Northern blot analysis documented that two transcription factor genes chosen for further study, c-myc promoter-binding protein (MBP-1) and X-box binding protein 1 (XBP-1), were up-regulated in U266 cells about 3-fold relative to the cell cycle-dependent beta-actin gene 12 h after IL-6 treatment"</p>
<p>SET CellLine = "U266"</p>
<p>SET Species = "9606"</p>
<div class="highlight"><pre><span></span><code>// disambiguation MBP-1 = HNGC ENO1
proteinAbundance(HGNC:IL6) increases rnaAbundance(HGNC:ENO1)
proteinAbundance(HGNC:IL6) increases rnaAbundance(HGNC:XBP1)
</code></pre></div>

<h4 id="short-form">Short Form<a class="headerlink" href="#short-form" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code>p(HGNC:IL6) -&gt; r(HGNC:ENO1)
p(HGNC:IL6) -&gt; r(HGNC:XBP1)
</code></pre></div>

<h4 id="causal-decrease">Causal decrease<a class="headerlink" href="#causal-decrease" title="Permanent link">&para;</a></h4>
<h4 id="example_4">Example<a class="headerlink" href="#example_4" title="Permanent link">&para;</a></h4>
<p>This statement demonstrates a causal statement using the <code>&lt;&lt;Xdecreases, decreases&gt;&gt;</code> relationship. The statement expresses that increases in the abundance of corticosteroid molecules cause decreases in the frequency or intensity of the biological process inflammation. This statement is annotated with an Anatomy and Disease to indicate that the relationship was observed in the context of the <em>cardiovascular system</em> and the disease <em>Stroke</em>.</p>
<h4 id="long-form_1">Long Form<a class="headerlink" href="#long-form_1" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code> SET Citation = {&quot;PubMed&quot;, &quot;J Mol Med. 2003 Mar;81(3):168-74. Epub 2003 Mar 14.&quot;, &quot;12682725&quot;}

 SET Support =&quot;high-dose steroid treatment decreases vascular
  inflammation and ischemic tissue damage after myocardial
  infarction and stroke through direct vascular effects involving
  the nontranscriptional activation of eNOS&quot;

 SET Anatomy = &quot;cardiovascular system&quot;

 SET MeSHDisease = &quot;Stroke&quot;

 abundance(CHEBI:corticosteroid) decreases biologicalProcess(MESHD:Inflammation)
</code></pre></div>

<h4 id="short-form_1">Short Form<a class="headerlink" href="#short-form_1" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code>a(CHEBI:corticosteroid) -| path(MESHD:Inflammation)
</code></pre></div>

<h4 id="causes-no-change">Causes no change<a class="headerlink" href="#causes-no-change" title="Permanent link">&para;</a></h4>
<p>The <code>&lt;&lt;Xcnc, causesNoChange&gt;&gt;</code> relationship can be used to record the lack of an observed effect.</p>
<h4 id="example_5">Example<a class="headerlink" href="#example_5" title="Permanent link">&para;</a></h4>
<p>The epidermal growth factor receptor (EGFR) ligand amphiregulin (AREG) is observed to increase NF-kappaB transcriptional activity while the EGFR ligand EGF has no effect.These statements express that an increase of AREG protein abundance causes an observed increase in the transcriptional activity of the NF-kappaB complex, and that an increase EGF does not.</p>
<h4 id="long-form_2">Long Form<a class="headerlink" href="#long-form_2" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code> SET Citation = {&quot;PubMed&quot;, &quot;Mol Cancer Res 2007 Aug 5(8) 847-61&quot;, &quot;17670913&quot;}

 SET Support =&quot;Furthermore, EGFR, activated by amphiregulin but not
  epidermal growth factor, results in the prompt activation of the
  transcription factor nuclear factor-kappaB (NF-kappaB)&quot;

 # disambiguation Amphiregulin = HGNC AREG

 proteinAbundance(HGNC:AREG) increases activity(complexAbundance(GOCC:&quot;NF-kappaB complex&quot;), molecularActivity(tscript))

 proteinAbundance(HGNC:EGF) causesNoChange activity(complexAbundance(GOCC:&quot;NF-kappaB complex&quot;), molecularActivity(tscript))
</code></pre></div>

<h4 id="short-form_2">Short Form<a class="headerlink" href="#short-form_2" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code>p(HGNC:AREG) -&gt; act(complex(GOCC:&quot;NF-kappaB complex&quot;), ma(tscript))
p(HGNC:EGF) causesNoChange act(complex(GOCC:&quot;NF-kappaB complex&quot;), ma(tscript))
</code></pre></div>

<h3 id="correlative-statement-examples">Correlative Statement Examples<a class="headerlink" href="#correlative-statement-examples" title="Permanent link">&para;</a></h3>
<p>&lt;&gt; link abundances and biological processes when no causal relationship is known.</p>
<ul>
<li>&lt;&gt;</li>
<li>&lt;&gt;</li>
</ul>
<h4 id="negativecorrelation">negativeCorrelation<a class="headerlink" href="#negativecorrelation" title="Permanent link">&para;</a></h4>
<p>This statement expresses that an increase in cytoplasmic FGF2 protein positively correlates with an increase in the pathology Chronic Obstructive Pulmonary Disease. The subject and object terms of correlative statements are interchangeable. The <code>&lt;&lt;XnegCor, negativeCorrelation&gt;&gt;</code> relationship is used to represent inverse correlative relationships, i.e., a decrease in A is correlated with an increase in B.</p>
<div class="highlight"><pre><span></span><code> SET Citation = {&quot;PubMed&quot;, &quot;J Pathol. 2005 May;206(1):28-38.&quot;, &quot;15772985&quot;}

 SET Support =&quot;Quantitative digital image analysis revealed
 increased cytoplasmic expression of FGF-2 in bronchial epithelium
 (0.35 +/- 0.03 vs 0.20 +/- 0.04, p &lt; 0.008) and nuclear
 localization in ASM (p &lt; 0.0001) in COPD patients compared with
 controls.&quot;

 SET Tissue = &quot;epithelium&quot;

 proteinAbundance(HGNC:FGF2, location(GOCC:cytoplasm)) positiveCorrelation \
  pathology(MESHD:&quot;Pulmonary Disease, Chronic Obstructive&quot;)
</code></pre></div>

<h4 id="association">association<a class="headerlink" href="#association" title="Permanent link">&para;</a></h4>
<p>The direction of causal effect or correlation of two abundance or biological process terms is not always specified. The <code>&lt;&lt;Xassociation, association&gt;&gt;</code> relationship can be used in these cases.</p>
<p>This statement represents that abundance of protein designated by the name Nr2f2 in the MGI namespace is associated in an unspecified manner with the biological process angiogenesis.</p>
<h4 id="long-form_3">Long Form<a class="headerlink" href="#long-form_3" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code> SET Citation = {&quot;PubMed&quot;, &quot;Mech Ageing Dev. 2004 Oct-Nov;125(10-11):719-32.&quot;, &quot;15541767&quot;}

 SET Support =&quot;COUP-TFII is involved in the angiogenic process in the developing embryos.&quot;

 # disambiguation - COUP-TFII refers to MGI Nr2f2

 SET MeSHAnatomy = &quot;Embryo, Mammalian&quot;

 proteinAbundance(MGI:Nr2f2) association biologicalProcess(GO:angiogenesis)
</code></pre></div>

<h4 id="short-form_3">Short Form<a class="headerlink" href="#short-form_3" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code>p(MGI:NR2F2) -- bp(GO:angiogenesis)
</code></pre></div>

<h3 id="direct-causal-statement-examples">Direct Causal Statement Examples<a class="headerlink" href="#direct-causal-statement-examples" title="Permanent link">&para;</a></h3>
<p>The following examples demonstrate the use of &lt;&gt; in causal statements. The direct causal relationships <code>&lt;&lt;XdIncreases, directlyIncreases&gt;&gt;</code> and <code>&lt;&lt;XdDecreases, directlyDecreases&gt;&gt;</code> are special forms of the causal <code>&lt;&lt;Xincreases, increases&gt;&gt;</code> and <code>&lt;&lt;Xdecreases, decreases&gt;&gt;</code> relationships where the mechanism of the causal relationship involves the physical interaction of entities related to the BEL Statement subject and object terms.</p>
<ul>
<li>&lt;&gt;</li>
<li>&lt;&gt;</li>
<li>&lt;&gt;</li>
<li>&lt;&gt;</li>
<li>&lt;&gt;</li>
</ul>
<h4 id="example-ligand-and-receptor">Example - Ligand and Receptor<a class="headerlink" href="#example-ligand-and-receptor" title="Permanent link">&para;</a></h4>
<p>In this example, the <code>&lt;&lt;XdIncreases, directlyIncreases&gt;&gt;</code> relationship is used to represent activation of a receptor by its ligand. This statement expresses that amphiregulin (AREG) activates its receptor, the Epidermal Growth Factor Receptor (EGFR). This relationship is direct because ligands directly interact with their receptors.</p>
<h4 id="long-form_4">Long Form<a class="headerlink" href="#long-form_4" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code>SET Citation = {&quot;PubMed&quot;, &quot;Mol Cancer Res 2007 Aug 5(8) 847-61&quot;, &quot;17670913&quot;}
SET Support =&quot;Furthermore, EGFR, activated by amphiregulin&quot;
// disambiguation Amphiregulin = HGNC AREG
// EGFR is known to have kinase activity
proteinAbundance(HGNC:AREG) directlyIncreases activity(proteinAbundance(HGNC:EGFR), molecularActivity(kin))
</code></pre></div>

<h4 id="short-form_4">Short Form<a class="headerlink" href="#short-form_4" title="Permanent link">&para;</a></h4>
<p>p(HGNC:AREG) =&gt; act(p(HGNC:EGFR), ma(kin))</p>
<h4 id="example-kinase-and-substrate">Example - Kinase and Substrate<a class="headerlink" href="#example-kinase-and-substrate" title="Permanent link">&para;</a></h4>
<p>In this example, the <code>&lt;&lt;XdIncreases, directlyIncreases&gt;&gt;</code> relationship is used to represent the phosphorylation of a protein substrate by a kinase. This statement expresses that the kinase activity of CDK1 protein causes an increase in the modification of FOXO1 protein by phosphorylation at serine 249. The relationship is direct because the kinase physically interacts with its target.</p>
<h4 id="long-form_5">Long Form<a class="headerlink" href="#long-form_5" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code>SET Citation = {&quot;PubMed&quot;, &quot;Science 2008 Mar 21 319(5870) 1665-8.&quot;, &quot;18356527&quot;}
SET Support =&quot;We found that Cdk1 phosphorylated the transcription factor FOXO1 at Ser249 in vitro and in vivo.&quot;
activity(p(HGNC:CDK1), ms(kin)) =&gt; p(HGNC:FOXO1, pmof(Ph, Ser, 249))
</code></pre></div>

<h4 id="example-catalyst-and-reaction">Example - Catalyst and Reaction<a class="headerlink" href="#example-catalyst-and-reaction" title="Permanent link">&para;</a></h4>
<p>In this example, the direct activation of a reaction by a catalytic enzyme is represented. The statement indicates that an increase in the catalytic activity of ALOX5 increase the transformation of the reactant '5(S)-HPETE' to the products 'leukotriene A4' and 'water'. The relationship is considered direct because ALOX5 protein is the catalyzing enzyme.</p>
<h4 id="long-form_6">Long Form<a class="headerlink" href="#long-form_6" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code> SET Citation = {&quot;Other&quot;, &quot;Reactome: Leukotriene synthesis&quot;, &quot;REACT_15354.1&quot;}

 SET Support =&quot;Dehydration of 5-HpETE to leukotriene A4. In the
  second step, 5-lipoxygenase converts 5-HpETE to an allylic
  epoxide, leukotriene A4.&quot;

 activity(proteinAbundance(HGNC:ALOX5), molecularActivity(cat)) directlyIncreases \
  reaction(reactants(abundance(CHEBI:&quot;5(S)-HPETE&quot;)), \
  products(abundance(CHEBI:&quot;leukotriene A4&quot;), abundance(CHEBI:water)))
</code></pre></div>

<h4 id="short-form_5">Short Form<a class="headerlink" href="#short-form_5" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code> act(p(HGNC:ALOX5), ma(cat)) =&gt; rxn(reactants(a(CHEBI:&quot;5(S)-HPETE&quot;)), products(a(CHEBI:&quot;leukotriene A4&quot;), a(CHEBI:water)))
</code></pre></div>

<h4 id="example-self-referential-relationships">Example - Self-Referential Relationships<a class="headerlink" href="#example-self-referential-relationships" title="Permanent link">&para;</a></h4>
<p>In this example, the <code>&lt;&lt;XdDecreases, directlyDecreases&gt;&gt;</code> relationship is used to represent the effect of a protein modification on the activity of the same protein. These statements express that the modification of GSK3A and GSK3B protein by phosphorylation on serines 9 and 21, respectively, inhibits the activity of GSK3A and GSK3B. These relationships are considered direct, because they are self-referential. The modification of the protein abundance by phosphorylation inhibits the activity of the same protein abundance.</p>
<h4 id="long-form_7">Long Form<a class="headerlink" href="#long-form_7" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code> SET Citation = {&quot;PubMed&quot;, &quot;Proc Natl Acad Sci U S A 2000 Oct 24 97(22) 11960-5&quot;, &quot;11035810&quot;}

 SET Support =&quot;GSK-3 activity is inhibited through phosphorylation
  of serine 21 in GSK-3 alpha and serine 9 in GSK-3 beta.&quot;

 proteinAbundance(HGNC:GSK3A, proteinModification(Ph, Ser, 21)) \
  directlyDecreases activity(proteinAbundance(HGNC:GSK3A))

 proteinAbundance(HGNC:GSK3B, proteinModification(Ph, Ser, 9)) \
  directlyDecreases activity(proteinAbundance(HGNC:GSK3B))
</code></pre></div>

<h4 id="short-form_6">Short Form<a class="headerlink" href="#short-form_6" title="Permanent link">&para;</a></h4>
<div class="highlight"><pre><span></span><code>p(HGNC:GSK3A, pmod(Ph, S, 21)) =| act(p(HGNC:GSK3A))
p(HGNC:GSK3B, pmod(Ph, S, 9)) =| act(p(HGNC:GSK3B))
</code></pre></div>

<h4 id="example-direct-transcriptional-control">Example - Direct Transcriptional Control<a class="headerlink" href="#example-direct-transcriptional-control" title="Permanent link">&para;</a></h4>
<p>In this example, the direct activation of a RNA transcription is encoded. The statement expresses that increases in the transcriptional activity of FOXO1 protein directly increase the RNA abundance of CEBPB. This relationship is considered direct because the transcription factor, FOXO1, directly binds the promoter of the CEBPB gene, increasing the expression of CEBPB RNA.</p>
<h4 id="long-form_8">Long Form<a class="headerlink" href="#long-form_8" title="Permanent link">&para;</a></h4>
<p>SET Citation = {"PubMed", "Biochem Biophys Res Commun. 2009 Jan 9;378(2):290-5. Epub 2008 Nov 21.", "19026986"}</p>
<p>SET Support ="We found that Foxo1 increased the expression of CCAAT/enhancer binding protein (C/EBPbeta, a positive regulator of monocyte chemoattractant protein (MCP)-1 and interleukin (IL)-6 genes, through directly binding to its promoter."</p>
<p>activity(proteinAbundance(HGNC:FOXO1), molecularActivity(tscript))  directlyIncreases rnaAbundance(HGNC:CEBPB)</p>
<h4 id="short-form_7">Short Form<a class="headerlink" href="#short-form_7" title="Permanent link">&para;</a></h4>
<p>act(p(HGNC:FOXO1), ma(tscript)) =&gt; r(HGNC:CEBPB)</p>
<h3 id="nested-statement-example">Nested Statement Example<a class="headerlink" href="#nested-statement-example" title="Permanent link">&para;</a></h3>
<p>This example demonstrates use of a nested causal statement in which the object of a causal statement is itself a causal statement. In the relationship described by the evidence text, CLSPN specifically increases the activity of ATR to phosphorylate the target protein CHEK1 and does not affect the kinase activity of ATR towards its other targets. The use of the nested statement allows the representation of the information that CLSPN increases the phosphorylation of CHEK1 via the kinase activity of ATR, without incorrectly indicating that CLSPN generally increases the kinase activity of ATR.</p>
<h4 id="long-form_9">Long Form<a class="headerlink" href="#long-form_9" title="Permanent link">&para;</a></h4>
<p>SET Citation = {"PubMed", "Mol Cell Biol 2006 Aug 26(16) 6056-64.", "16880517"}</p>
<p>SET Species = "9606"</p>
<p>SET Support ="Consistently, the RNAi-mediated ablation of Claspin selectively abrogated ATR's ability to phosphorylate Chk1 but not other ATR targets."</p>
<p>proteinAbundance(HGNC:CLSPN) increases  (activity(proteinAbundance(HGNC:ATR), molecularActivity(kin)) directlyIncreases proteinAbundance(HGNC:CHEK1, proteinModification(Ph)))</p>
<h4 id="short-form_8">Short Form<a class="headerlink" href="#short-form_8" title="Permanent link">&para;</a></h4>
<p>p(HGNC:CLSPN) -&gt; (act(p(HGNC:ATR), ma(kin)) =&gt; p(HGNC:CHEK1, pmod(Ph)))</p>
<h2 id="other-examples">Other Examples<a class="headerlink" href="#other-examples" title="Permanent link">&para;</a></h2>
<h3 id="membership-assignment-examples">Membership Assignment Examples<a class="headerlink" href="#membership-assignment-examples" title="Permanent link">&para;</a></h3>
<p>These examples demonstrate the assignment of members to groups. Because all BEL terms denote classes, membership in a group is an important special case where subsets of a class that define the class are designated.</p>
<p>[TIP]</p>
<p>The BEL Framework adds family members to protein families and complex components to named complexes during network compilation.</p>
<ul>
<li>&lt;&gt;</li>
<li>&lt;&gt;</li>
</ul>
<h4 id="protein-family">Protein Family<a class="headerlink" href="#protein-family" title="Permanent link">&para;</a></h4>
<p>In this example, members of a protein family are assigned using the <code>&lt;&lt;hasMember&gt;&gt;</code> and <code>&lt;&lt;hasMembers&gt;&gt;</code> relationships.</p>
<p>The <code>hasMembers</code> relationship is used to assign a list of protein abundances as members of a protein family. This relationship is a syntactic convenience that is equivalent to the set of two statements using the <code>hasMember</code> relationship. These statements designate the protein abundances of MAPK8 and MAPK9 as members of the JNK MAPK protein family. The term representing the JNK family is a protein abundance based on the name <em>MAPK JNK Family</em> in the Selventa Protein Families namespace. p(SFAM:"MAPK JNK Family") hasMembers list(p(HGNC:MAPK8), p(HGNC:MAPK9))</p>
<p>The <code>hasMember</code> relationship is used to assign individual protein abundances to a protein family. p(SFAM:"MAPK JNK Family") hasMember p(HGNC:MAPK8) p(SFAM:"MAPK JNK Family") hasMember p(HGNC:MAPK9)</p>
<h4 id="complex-component">Complex Component<a class="headerlink" href="#complex-component" title="Permanent link">&para;</a></h4>
<p>In this example components are assigned to a named protein complex using the &lt;&gt; and &lt;&gt; relationships.</p>
<p>The <code>hasComponents</code> relationship is similar to the <code>hasMembers</code> relationship and is used to assign a list of abundances as components of a complex.These statements designate the protein abundances of RAD9A, RAD1, and HUS1 as components of the complex abundance of the <em>checkpoint clamp complex</em>.</p>
<p>complex(GOCC:"checkpoint clamp complex") hasComponents list(p(HGNC:RAD9A), p(HGNC:RAD1), p(HGNC:HUS1))</p>
<p>The <code>hasComponent</code> relationship is used to assign individual abundances to a named protein complex.</p>
<p>complex(GOCC:"checkpoint clamp complex") hasComponent p(HGNC:RAD9A)</p>
<p>complex(GOCC:"checkpoint clamp complex") hasComponent p(HGNC:RAD1)</p>
<p>complex(GOCC:"checkpoint clamp complex") hasComponent p(HGNC:HUS1)</p>
<p>The single <code>hasComponents</code> statement is equivalent to the set of three <code>hasComponent</code> statements.</p>
<h2 id="example-binding-interaction">Example - Binding Interaction<a class="headerlink" href="#example-binding-interaction" title="Permanent link">&para;</a></h2>
<p>The <code>complexAbundance()</code> function can be used to specify molecular interactions between abundances. This function can take either a list of abundances that define a molecular complex or a namespace value that represents a molecular complex (e.g., many GO Cellular Component values) as an argument. These examples demonstrate the use of the <code>complexAbundance()</code> function to represent protein-protein, protein-chemical, and protein-DNA interactions.</p>
<ul>
<li>Protein – protein interactions</li>
<li>Protein – DNA interactions</li>
<li>Protein – small molecule interactions</li>
</ul>
<h2 id="example-protein-protein-interaction-as-bel-statement">Example - protein-protein interaction as BEL statement<a class="headerlink" href="#example-protein-protein-interaction-as-bel-statement" title="Permanent link">&para;</a></h2>
<p>This statement represents that MTOR and AKT1S1 proteins physically interact. Note that this statement has only an object term and no subject term and relationship.</p>
<div class="highlight"><pre><span></span><code>SET Citation = {&quot;PubMed&quot;, &quot;17277771&quot;}
SET Support = &quot;Here, we identify PRAS40 (proline-rich Akt/PKB substrate 40 kDa) as a novel mTOR binding partner&quot;
// disambiguation PRAS40 = HGNC AKT1S1
complex(p(hgnc:391 ! AKT1S1), p(hgnc:3942 ! MTOR))
</code></pre></div>

<h2 id="example-protein-protein-interaction-as-statement-object">Example - protein-protein interaction as Statement object<a class="headerlink" href="#example-protein-protein-interaction-as-statement-object" title="Permanent link">&para;</a></h2>
<p>Here, a protein-protein interaction is the object of a BEL Statement. This statement expresses that the MTOR and STAT3 proteins associate and that increases in the protein abundance of BMP4 can increase the abundance of the complex comprised of MTOR and STAT3.</p>
<div class="highlight"><pre><span></span><code>SET Citation = {&quot;PubMed&quot;, &quot;12796477&quot;}
SET Support = &quot;Upon BMP4 treatment, the serine-threonine kinase FKBP12/rapamycin-associated protein (FRAP), mammalian target of rapamycin (mTOR), associates with Stat3 and facilitates STAT activation.&quot;
p(hgnc:1071 ! BMP4) -&gt; complex(p(hgnc:3942 ! MTOR), p(hgnc:11364 ! STAT3))
</code></pre></div>

<h4 id="protein-dna-interactions">Protein – DNA interactions<a class="headerlink" href="#protein-dna-interactions" title="Permanent link">&para;</a></h4>
<h2 id="example-transcription-factor-protein-binding-to-dna">Example - transcription factor protein binding to DNA<a class="headerlink" href="#example-transcription-factor-protein-binding-to-dna" title="Permanent link">&para;</a></h2>
<p>This statement expresses that STAT3 protein binds to the CCL11 gene DNA, and that this association is increased by IL17A.</p>
<div class="highlight"><pre><span></span><code>SET Citation = {&quot;PubMed&quot;, &quot;19265112&quot;}
SET Support =&quot;IL-17A induced at 1 h a marked enrichment of STAT3- associated CCL11 promoter DNA&quot;
p(hgnc:5981 ! IL17A) -&gt; complex(p(hgnc:11364 ! STAT3), g(hgnc:10610 ! CCL11))
</code></pre></div>

<h4 id="protein-small-molecule-interactions">Protein – small molecule interactions<a class="headerlink" href="#protein-small-molecule-interactions" title="Permanent link">&para;</a></h4>
<h2 id="example-protein-binding-to-a-small-molecule">Example - protein binding to a small molecule<a class="headerlink" href="#example-protein-binding-to-a-small-molecule" title="Permanent link">&para;</a></h2>
<p>This statement represents that PIP3 binds AKT1 protein.</p>
<div class="highlight"><pre><span></span><code>SET Citation = {&quot;PubMed&quot;, &quot;15987444&quot;}
SET Support = &quot;After PIP3 binding, Akt1 is activated&quot;
// disambiguation PIP3 = CHEBI 1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate
complex(a(CHEBI:&quot;1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate&quot;), p(hgnc:391 ! AKT1))
</code></pre></div>

<h3 id="causal_1">Causal<a class="headerlink" href="#causal_1" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>p(HGNC:CCND1) =&gt; act(p(HGNC:CDK4))
</code></pre></div>

<p>The abundance of the protein designated by <strong>CCND1</strong> in the HGNC namespace directly increases the activity of the abundance of the protein designated by <strong>CDK4</strong> in the HGNC namespace.</p>
<h3 id="causal_2">Causal<a class="headerlink" href="#causal_2" title="Permanent link">&para;</a></h3>
<div class="highlight"><pre><span></span><code>p(HGNC:BCL2)-| bp(MESHPP:Apoptosis)
</code></pre></div>

<p>The abundance of the protein designated by <strong>BCL2</strong> in the HGNC namespace decreases the biological process designated by <strong>apoptosis</strong> in the MESHPP (phenomena and processes) namespace.</p>
                
              
              
                


              
            </article>
          </div>
        </div>
      </main>
      
        
<footer class="md-footer">
  
    <div class="md-footer-nav">
      <nav class="md-footer-nav__inner md-grid" aria-label="Footer">
        
          <a href="../annotations/" title="Annotations" class="md-footer-nav__link md-footer-nav__link--prev" rel="prev">
            <div class="md-footer-nav__button md-icon">
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
            </div>
            <div class="md-footer-nav__title">
              <div class="md-ellipsis">
                <span class="md-footer-nav__direction">
                  Previous
                </span>
                Annotations
              </div>
            </div>
          </a>
        
        
          <a href="../changelog/" title="Changelog" class="md-footer-nav__link md-footer-nav__link--next" rel="next">
            <div class="md-footer-nav__title">
              <div class="md-ellipsis">
                <span class="md-footer-nav__direction">
                  Next
                </span>
                Changelog
              </div>
            </div>
            <div class="md-footer-nav__button md-icon">
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"/></svg>
            </div>
          </a>
        
      </nav>
    </div>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-footer-copyright">
        
        Made with
        <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
          Material for MkDocs
        </a>
      </div>
      
    </div>
  </div>
</footer>
      
    </div>
    
      <script src="../assets/javascripts/vendor.d710d30a.min.js"></script>
      <script src="../assets/javascripts/bundle.5f27aba8.min.js"></script><script id="__lang" type="application/json">{"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents"}</script>
      
      <script>
        app = initialize({
          base: "..",
          features: ["instant"],
          search: Object.assign({
            worker: "../assets/javascripts/worker/search.27c6a5e6.min.js"
          }, typeof search !== "undefined" && search)
        })
      </script>
      
    
  </body>
</html>